Started by timer
Running in Durability level: MAX_SURVIVABILITY
[8mha:////4DOlrZhma2TKYqx8mtRBwGXVE8Xt4z3OJyo/zi+ICDdzAAAAoh+LCAAAAAAAAP9tjTEOwjAQBM8BClpKHuFItIiK1krDC0x8GCfWnbEdkooX8TX+gCESFVvtrLSa5wtWKcKBo5UdUu8otU4GP9jS5Mixv3geZcdn2TIl9igbHBs2eJyx4YwwR1SwULBGaj0nRzbDRnX6rmuvydanHMu2V1A5c4MHCFXMWcf8hSnC9jqYxPTz/BXAFEIGsfuclm8zQVqFvQAAAA==[0m[Pipeline] Start of Pipeline
[8mha:////4B4K7nOhvaaLmbeE3w0NEhE1GAALwLDjakw9plXgkY6XAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycohUghExsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jduZBmjwAAAAA==[0m[Pipeline] node
Running on [8mha:////4C8PgMHPegRpCg8+HpIiIHK02uxvuQktJylkqRHkJagNAAAAox+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLgAzeEgYJ/eT83ILSktQiICOvpCg/Jye1SNfSUh8ABjlyb80AAAA=[0mcontroller-99 in /root/.jenkins/workspace/cyp-test-nic-sriov-5times
[8mha:////4NUxBMzN6CtFAbQ4pdo7pH8ZuwVHigLf0SdP3FcjQTwyAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0gA0xsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jfoP95RwAAAAA==[0m[Pipeline] {
[8mha:////4OCB1+Up79PmV3LnMeomKr9r+JFCKkFjAfTpr2UorbLOAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0gQkxsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jc09154wAAAAA==[0m[Pipeline] stage
[8mha:////4N6KQN35K8CT/Pcr9a+R4oOEu4uEh3phvpB1t67zd2hmAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0ggUxsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jek7ggRwAAAAA==[0m[Pipeline] { (rm project dir)
[8mha:////4EWjgIQS1R18e39T6LdOu3ZpSMSXMburmKwxJHd8e163AAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0gwExsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jcChmMxwAAAAA==[0m[Pipeline] script
[8mha:////4NOJ/YVWDlnnBpjDdLKlLZkF5LrR387vblsx7fWhZkPlAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0BAkxsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jfpX/cvwAAAAA==[0m[Pipeline] {
[8mha:////4MAm0moKTc8Eo2PcB0LgkHNHI7Xtf5Poej2Mf/l8xh/lAAAAoh+LCAAAAAAAAP9tjTEOAiEURD9rLGwtPQTbaoyVLaHxBMgiwpL/WWDdrTyRV/MOEjexcpJJ5k3zXm9Y5wQnSpZ7g73DrB2PYbR18YlSfws0cU9XrgkzBcOlmSR15rygpGJgCWtgJWBjUAfKDm2BrfDqodqg0LaXkup3FNC4boAnMFHNRaXyhTnB7j52mfDn+SuAORZg+9pD/AAeoCAqvAAAAA==[0m[Pipeline] sh
+ ls /root/.jenkins/workspace/cyp-test-nic-sriov-5times/npxtester
1.0.0.0001.version
conf
installation
libs
log_analyzer
README.md
testconf
testLibs
Tests
[8mha:////4F3BY6hez6Yn9lSF4dYULOlz4RwK0Fdsp1OYWtj0vxDtAAAAox+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMKBDglR0VppeIFJjHFi3QX7QlLxIr7GH4iIRMVWO9PM641lijhydKqx1HpKlVdd6N301MCxvQYeVMMXVTElDlaVdii5tqcZSxaLeVmOhcbKUhU4eXKCtW7MwxTBkCvOEid30Mh9fccTmZ7KYqJ8YYzY3Po6Mf06fwMYO0G2F+S7bfcBITL9lL0AAAA=[0m[Pipeline] echo
dir is exists and rm dir
[8mha:////4KL1cTFlC/t8/0LRf71hJwxXIUFLIMviWtcIHXR2W8epAAAAox+LCAAAAAAAAP9tjbEOgjAURS8YB1dHP6KEzcQ4uTYsfkGFWgvNe9g+hMkv8tf8B4kkTt7pnrOc1xvrFHHk6FRrqfOUaq/6MLj5qZFjdw08qpYvqmZKHKyq7FhxY08LViwWy7IcK42NpTpw8uQEW92ahymCIVecJc7uoJH75o4nMj2XxUT5whSxuw1NYvp1/gYw9YJsL8jLsv8AYAPmjb0AAAA=[0m[Pipeline] sh
+ rm -rf /root/.jenkins/workspace/cyp-test-nic-sriov-5times/npxtester
[8mha:////4GzJynz9IX3CLl34e/1JpYOeLz7FDsqMtIihCtFGqNpZAAAAox+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMIRVEiIitZKwwtMYowT6y7YF5KKF/E1/kBEJCq22plmXm8sU8SRo1ONpdZTqrzqQu+mpwaO7TXwoBq+qIopcbCqtEPJtT3NWLJYzMtyLDRWlqrAyZMTrHVjHqYIhlxxlji5g0bu6zueyPRUFhPlC2PE5tbXienX+RvA2HWCfLsTZPsPzjQzob0AAAA=[0m[Pipeline] }
[8mha:////4MvApYhNLg1xEFG2iXyHCog7Z9h4MbdAsNSDoK4wJnDgAAAApB+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMIRoqBAVLRWGl5gEmOcWHfBvpBUvIiv8QciIlGx1c4083pjmSKOHJ1qLLWeUuVVF3o3PTVwbK+BB9XwRVVMiYNVpR1Kru1pxpLFYl6WY6GxslQFTp6cYK0b8zBFMOSKs8TJHTRyX9/xRKanspgoXxgjNre+Tky/zt8Axq4T5NudINt/AO8A7Ay9AAAA[0m[Pipeline] // script
[8mha:////4P1g47DHQeYIMdAQPrqVFJW5XTEYSJkMqP0k+p1upCWxAAAApB+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMIREqJBVLRWGl5gEmOcWHfBvpBUvIiv8QciIlGx1c4083pjmSKOHJ1qLLWeUuVVF3o3PTVwbK+BB9XwRVVMiYNVpR1Kru1pxpLFYl6WY6GxslQFTp6cYK0b8zBFMOSKs8TJHTRyX9/xRKanspgoXxgjNre+Tky/zt8Axq4T5NudINt/AGnsy8m9AAAA[0m[Pipeline] }
[8mha:////4Lc6WrLYEiiAC7rTWY8q4RQPGu1FD+hrgClvEqpzcg2sAAAAox+LCAAAAAAAAP9tjTESgjAQRT84FraWHiKMBZVjZZuh8QQRYgxkdjFZhMoTeTXvICMzVv7qv9e81xvrFHHk6FRrqfOUaq/6MLj5qZFjdw08qpYvqmZKHKyq7FhxY08LViwWy7IcK42NpTpw8uQEW92ahymCIVecJc7uoJH75o4nMj2XxUT5whSxuw1NYvp1/gYw9b0g35eCrPwAY5Sibb0AAAA=[0m[Pipeline] // stage
[8mha:////4LPzYIRsJlP9vBl87WR2FbvV5590n9jZkqHCRBijPEs5AAAApx+LCAAAAAAAAP9tjTEOwjAQBM9BKWgpeYQjJESDqGgtN7zAxMY4se6CfSGpeBFf4w8EIlGx1c5Kq3m+oMwJDpS8bBy2AXMdZBd7PzU5UGovkQbZ0FnWhJmik9oNmqw7zqiJHcwRBSwULB3WkXJAz7BSjbmbKhr01YnTtO0VFMHe4AFCTWY2ib8wJlhfe5sJf56/Ahg7BrFlKDa7z7F8A6sHc4fBAAAA[0m[Pipeline] stage
[8mha:////4P0yjWfQiEwQxZO7i9obwDXrYwQclNR3i//1CTGbFwBFAAAApx+LCAAAAAAAAP9tjTEOwjAUQ3+LOrAycohULDAgpq5RFk4QmhDSRv+Xn5R24kRcjTvQUokJD5ZtyXqvNxSR4UTsRGOx9RhrL7rQuymJgbi9BhpEQxdRE0YKVig7KDK2WqqiZGFRlsNKwtpiHSh6dAk2stEPXQaNrjwnnrajhNybOzwhkxM5aU7fMjJsb72JhD/OXwCMXYJ8t5/tMD+LD1+IXOLCAAAA[0m[Pipeline] { (git clone)
[8mha:////4I3tKNYAz7At397ZkNOs6SFwEn2bzn3ZBhsoegW1dhRwAAAApB+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIiKpAQVVrLDS8wsTFOrDtjOyQVL+Jr/IFAJCq2WO1sM88XLFOEI0eLraHOUWocBt/baeHAsbt4HrDlMzZMib1BaQbJ2tQzSs4G5hQlLASsDDWekyObYS1adVeVV2SrU47TdxBQOn2DBxRiMmcV8xfGCJtrrxPTz/NXAGPIUG53n9qHNxXfVKy+AAAA[0m[Pipeline] sh
+ git clone -b develop https://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@github.com/Jiz1028/applications.validation.network-platform.npxtester.git npxtester
Cloning into 'npxtester'...
[8mha:////4CRWXV91kUhqh2bTVXI+2n5U32cQ89nJ7yub5blKQ29bAAAApB+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIiKoQQVVrLDS8wsTFOrDtjOyQVL+Jr/IFAJCq2WO1sM88XLFOEI0eLraHOUWocBt/baeHAsbt4HrDlMzZMib1BaQbJ2tQzSs4G5hQlLASsDDWekyObYS1adVeVV2SrU47TdxBQOn2DBxRiMmcV8xfGCJtrrxPTz/NXAGMIGcrt/lO7N9Jjga++AAAA[0m[Pipeline] }
[8mha:////4I4ZVrOFMvh2sVfxnfT9s6+paGHoH9OzkxEwdR4h+nRDAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOICFDQoFa3lhheYxBgn1l2wHZyKF/E1/oBFJCq2WO1sM683LIOHmr3BTlNvKTQWBzeavDCx76+OE3Z8wYYpsNModZLc6tOMkqOGOUUJCwErTY3jYMlEWItOPVTlFJnqHH3+jgJK297hCYXI5qh8/MLkYXMb28D08/wVwDQMEcr9Ntfu8AERGGHOvgAAAA==[0m[Pipeline] // stage
[8mha:////4J4BpfnRrLE6odVrNZGgZ11wIRy0HYHBalvcHAcD12j/AAAAph+LCAAAAAAAAP9tjTEOwjAQBM9BKWgpeYQjEB2iorXc8AITG+PEugv2haTiRXyNPxCIRMVWOyut5vmCMic4UPKycdgGzHWQXez91ORAqb1EGmRDZ1kTZopOajdosu44oyZ2MEcUsFCwdFhHygE9w0o15m6qaNBXJ07TtldQBHuDBwg1mdkk/sKYYH3tbSb8ef4KYOwYxI6h2G4+x/INji2eq8EAAAA=[0m[Pipeline] stage
[8mha:////4AgPhGmio4pMNzq5gItmuDxswb3P5dwyIofUIBdMXBTmAAAAph+LCAAAAAAAAP9tjTEOwjAUQ3+DOrAycohUdEVMrFEXThCaEJJG/7dJSjpxIq7GHWipxIQHy7ZkvdcbyhjgRMFwp7GzGFvLez+aOfFMobt5ytzRlbeEkbzmjc4NKX1ea0NJw6qCwUbAVmPrKVo0CXbCyYesvERTXVKYt6MAZtUATyjETE4ypG+ZAuzvo4qEP85fAEx9AlYfFquXZ/kBAsTc3cIAAAA=[0m[Pipeline] { (pre-exec-set-args)
[8mha:////4G9dxluK5ut2lxdfo2JVi6OIov1ZH/881VUFzuPzanonAAAAox+LCAAAAAAAAP9tjTESwiAURH/iWNhaeggyE0vHypah8QSYIEKY/wkQSeWJvJp3EM2MlVvs7NvmPV+wjgGOFDSzCgeDsTPMu0mXxTKF4eooM0sX1hFGcooJlQX16rSgoKRgSVXDisNGYecoGtQJttzKu2ycRN2cUyjfgUNt+hEeUPFiTjKkL8wBdrepj4Q/z18BzD5B3baf2vs3w5BRkb4AAAA=[0m[Pipeline] sh
+ python /root/.jenkins/workspace/cyp-test-nic-sriov-5times/npxtester/conf/change_args.py '{"drive_info":{"drive_version":"1.6.4"},"dpdk_dir":"/opt/APP/utility/DPDK_install/dpdk-21.02/","vm_image_conf":{"image1":"uvm1.img","image2":"uvm2.img","image3":"rhel84rt.img","image4":"rhel84rt2.img","image_dir":"/home/image"},"sut_conf":{"host":"10.239.182.209","pf1":"ens785f2","os":"RHEL"},"pkg_generator":{"host":"10.239.182.79","pf1":"ens787f2"}}'
[8mha:////4FhUQONKPXFSeOjFO92QKbtxv93cUzSuhB7hx3xoWX6KAAAApB+LCAAAAAAAAP9tjbEOwiAURV9rHFwd/QiaNG7GyZWw+AXYIkLJexSodPKL/DX/QbSJk3e4uecu5/mCdQxwpKCZVTgYjJ1h3k26LJYpDFdHmVm6sI4wklNMqCyoV6cFBSUFS6oaVhw2CjtH0aBOsOVW3mXjJOrmnEL5Dhxq04/wgIoXc5IhfWEOsLtNfST8ef4KYPY+Qd3uP9W+Ad3NsHm+AAAA[0m[Pipeline] }
[8mha:////4EpM1Tihe33kOTBucrClpdwMCq1FF6wGwTU/AEOZmYaNAAAApB+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIikKgQVVrLDS8wsTFOrDtjOyQVL+Jr/IFAJCq2WO1sM88XLFOEI0eLraHOUWocBt/baeHAsbt4HrDlMzZMib1BaQbJ2tQzSs4G5hQlLASsDDWekyObYS1adVeVV2SrU47TdxBQOn2DBxRiMmcV8xfGCJtrrxPTz/NXAGMIGcrd/lPbN8JP5Su+AAAA[0m[Pipeline] // stage
[8mha:////4JJk4NQMkVX21N+ktknOqKLQw+FD2VU/fvD8dhBPuambAAAApx+LCAAAAAAAAP9tjTEOwjAQBM9BKWgpeYQjIUSDqGgtN7zAxMY4se6CfSGpeBFf4w8EIlGx1c5Kq3m+oMwJDpS8bBy2AXMdZBd7PzU5UGovkQbZ0FnWhJmik9oNmqw7zqiJHcwRBSwULB3WkXJAz7BSjbmbKhr01YnTtO0VFMHe4AFCTWY2ib8wJlhfe5sJf56/Ahg7BrFlKDa7z7F8AzYdm7bBAAAA[0m[Pipeline] stage
[8mha:////4GToWcz1lEtttcc1F67+obVKf97ChyGWoNBgdNb8ln6iAAAAqB+LCAAAAAAAAP9tjTEOwjAUQ3+LOrAycohUYoABMXWNsnCC0ISQNvq//KS0EyfiatyBlkpMeLBsS9Z7vaGIDCdiJxqLrcdYe9GF3k1JDMTtNdAgGrqImjBSsELZQZGx1VIVJQuLshxWEtYW60DRo0uwkY1+6DJodOU58bQdJeTe3OEJmZzISXP6lpFhe+tNJPxx/gJg7BLku/1sh/lZfAAyQCqkwgAAAA==[0m[Pipeline] { (run nic case)
[8mha:////4M/BBJOrvLliK1a64sS7A0hf4H4seiclg8D1IX9OqKAVAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIi0YCEqNJabniBiY1xYt0Z2yGpeBFf4w8EIlGxxWpnm3m+YJkiHDlabA11jlLjMPjeTgsHjt3F84Atn7FhSuwNSjNI1qaeUXI2MKcoYSFgZajxnBzZDGvRqruqvCJbnXKcvoOA0ukbPKAQkzmrmL8wRthce52Yfp6/AhhDhnK7+9Q+vAGvZp0gvgAAAA==[0m[Pipeline] sh
+ pytest /root/.jenkins/workspace/cyp-test-nic-sriov-5times/npxtester/Tests/testNics -s -m nicSriov5times --capture=tee-sys --html=/root/testconfig/testlogs/HtmlReport/nic_sriov5times_case_report.html --self-contained-html
============================= test session starts ==============================
platform linux -- Python 3.6.8, pytest-6.2.4, py-1.11.0, pluggy-0.13.1
rootdir: /root/.jenkins/workspace/cyp-test-nic-sriov-5times/npxtester/Tests/testNics, configfile: pytest.ini
plugins: metadata-1.11.0, html-3.1.1
collected 17 items / 16 deselected / 1 selected

npxtester/Tests/testNics/test_network_sriov_stress.py 2022-03-11 02:52:42 - NicTestFixture.py - [line:28] - root - INFO - ============================ Starting to run test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest[uvm1.img-uvm2.img] case ============================
2022-03-11 02:52:42 - test_network_sriov_stress.py - [line:28] - root - INFO - SSH SUT and setup configurations in the setup function.
2022-03-11 02:52:42 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-03-11 02:52:42 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-03-11 02:52:42 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.209
2022-03-11 02:52:42 - sftp.py - [line:158] - paramiko.transport.sftp - INFO - [chan 1] Opened sftp connection (server version 3)
2022-03-11 02:52:42 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-03-11 02:52:42 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-03-11 02:52:43 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.79
2022-03-11 02:52:46 - sshInstance.py - [line:46] - root - INFO - modinfo ice | grep ^version:
2022-03-11 02:52:46 - sshInstance.py - [line:50] - root - INFO - version:        1.6.4

2022-03-11 02:52:54 - sshInstance.py - [line:46] - root - INFO - rmmod ice
2022-03-11 02:53:06 - sshInstance.py - [line:50] - root - INFO - 
2022-03-11 02:53:14 - sshInstance.py - [line:46] - root - INFO - modprobe ice
2022-03-11 02:53:19 - sshInstance.py - [line:50] - root - INFO - 
2022-03-11 02:53:27 - sshInstance.py - [line:46] - root - INFO - ifconfig ens785f2 up
2022-03-11 02:53:27 - sshInstance.py - [line:50] - root - INFO - 
2022-03-11 02:53:35 - sshInstance.py - [line:46] - root - INFO - modinfo ice
2022-03-11 02:53:35 - sshInstance.py - [line:50] - root - INFO - filename:       /lib/modules/4.18.0-193.19.1.rt13.70.el8_2.x86_64/updates/drivers/net/ethernet/intel/ice/ice.ko
firmware:       intel/ice/ddp/ice.pkg
version:        1.6.4
license:        GPL v2
description:    Intel(R) Ethernet Connection E800 Series Linux Driver
author:         Intel Corporation, <linux.nics@intel.com>
rhelversion:    8.2
srcversion:     00C5667CA1FBE608C371B1C
alias:          pci:v00008086d0000151Dsv*sd*bc*sc*i*
alias:          pci:v00008086d0000124Fsv*sd*bc*sc*i*
alias:          pci:v00008086d0000124Esv*sd*bc*sc*i*
alias:          pci:v00008086d0000124Dsv*sd*bc*sc*i*
alias:          pci:v00008086d0000124Csv*sd*bc*sc*i*
alias:          pci:v00008086d0000189Asv*sd*bc*sc*i*
alias:          pci:v00008086d00001899sv*sd*bc*sc*i*
alias:          pci:v00008086d00001898sv*sd*bc*sc*i*
alias:          pci:v00008086d00001897sv*sd*bc*sc*i*
alias:          pci:v00008086d00001894sv*sd*bc*sc*i*
alias:          pci:v00008086d00001893sv*sd*bc*sc*i*
alias:          pci:v00008086d00001892sv*sd*bc*sc*i*
alias:          pci:v00008086d00001891sv*sd*bc*sc*i*
alias:          pci:v00008086d00001890sv*sd*bc*sc*i*
alias:          pci:v00008086d0000188Esv*sd*bc*sc*i*
alias:          pci:v00008086d0000188Dsv*sd*bc*sc*i*
alias:          pci:v00008086d0000188Csv*sd*bc*sc*i*
alias:          pci:v00008086d0000188Bsv*sd*bc*sc*i*
alias:          pci:v00008086d0000188Asv*sd*bc*sc*i*
alias:          pci:v00008086d0000159Bsv*sd*bc*sc*i*
alias:          pci:v00008086d0000159Asv*sd*bc*sc*i*
alias:          pci:v00008086d00001599sv*sd*bc*sc*i*
alias:          pci:v00008086d00001593sv*sd*bc*sc*i*
alias:          pci:v00008086d00001592sv*sd*bc*sc*i*
alias:          pci:v00008086d00001591sv*sd*bc*sc*i*
depends:        
name:           ice
vermagic:       4.18.0-193.19.1.rt13.70.el8_2.x86_64 SMP preempt mod_unload modversions 
parm:           debug:netif level (0=none,...,16=all) (int)
parm:           fwlog_level:FW event level to log. All levels <= to the specified value are enabled. Values: 0=none, 1=error, 2=warning, 3=normal, 4=verbose. Invalid values: >=5
 (ushort)
parm:           fwlog_events:FW events to log (32-bit mask)
 (ulong)

2022-03-11 02:53:43 - sshInstance.py - [line:46] - root - INFO - ifconfig ens785f2
2022-03-11 02:53:43 - sshInstance.py - [line:50] - root - INFO - ens785f2: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::42a6:b7ff:fe72:ccca  prefixlen 64  scopeid 0x20<link>
        ether 40:a6:b7:72:cc:ca  txqueuelen 1000  (Ethernet)
        RX packets 2  bytes 684 (684.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 16  bytes 2134 (2.0 KiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-03-11 02:53:51 - sshInstance.py - [line:46] - root - INFO - ifconfig ens787f2
2022-03-11 02:53:51 - sshInstance.py - [line:50] - root - INFO - ens787f2: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.10.11  netmask 255.255.255.0  broadcast 192.168.10.255
        inet6 fe80::42a6:b7ff:fe82:eb6  prefixlen 64  scopeid 0x20<link>
        ether 40:a6:b7:82:0e:b6  txqueuelen 1000  (Ethernet)
        RX packets 9750596304  bytes 14758031159546 (13.4 TiB)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 443416941  bytes 31595748310 (29.4 GiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-03-11 02:53:56 - test_network_sriov_stress.py - [line:389] - root - INFO - 

********************************************************Stress Disable and Enable for 1 time when VF is Assigned VM guest.*************************************************************


2022-03-11 02:53:59 - sshInstance.py - [line:46] - root - INFO - /opt/APP/utility/DPDK_install/dpdk-21.02/usertools/dpdk-devbind.py -s | grep ens785f2 | awk '{print $1;}'
2022-03-11 02:53:59 - sshInstance.py - [line:50] - root - INFO - 0000:4b:00.2

2022-03-11 02:54:04 - test_network_sriov_stress.py - [line:110] - root - INFO - pf_id was found: 0000:4b:00.2

2022-03-11 02:54:04 - test_network_sriov_stress.py - [line:397] - root - INFO - format_pf_id is 0000\:4b\:00.2

2022-03-11 02:54:04 - test_network_sriov_stress.py - [line:400] - root - INFO - pf_id_key: 4b
2022-03-11 02:54:10 - sshInstance.py - [line:94] - root - INFO - Activate the web console with: systemctl enable --now cockpit.socket
This system is not registered to Red Hat Insights. See https://cloud.redhat.com/
To register this system, run: insights-client --register
Last login: Fri Mar 11 01:38:00 2022 from 10.239.182.99
[root@npx-cyp7-rh82 ~]# ifconfig ens785f2 192.168.10.10
[root@npx-cyp7-rh82 ~]# 
2022-03-11 02:54:13 - test_network_sriov_stress.py - [line:301] - root - INFO - Successfully assigned ip to ethernet: ens785f2
2022-03-11 02:54:16 - sshInstance.py - [line:46] - root - INFO - echo 4 > /sys/class/net/ens785f2/device/sriov_numvfs
2022-03-11 02:54:17 - sshInstance.py - [line:50] - root - INFO - 
2022-03-11 02:54:22 - test_network_sriov_stress.py - [line:124] - root - INFO - Successfully Virtualized PF to VFs.
2022-03-11 02:54:25 - sshInstance.py - [line:46] - root - INFO -  lspci | grep "Virtual" | awk '{print $1;}'list
2022-03-11 02:54:25 - sshInstance.py - [line:50] - root - INFO - 4b:11.0
4b:11.1
4b:11.2
4b:11.3

2022-03-11 02:54:30 - test_network_sriov_stress.py - [line:138] - root - INFO - Succssfully Gotten vf_id list:

2022-03-11 02:54:33 - sshInstance.py - [line:46] - root - INFO - modprobe vfio-pci

2022-03-11 02:54:33 - sshInstance.py - [line:50] - root - INFO - 
2022-03-11 02:54:41 - sshInstance.py - [line:46] - root - INFO - /opt/APP/utility/DPDK_install/dpdk-21.02/usertools/dpdk-devbind.py -b vfio-pci 4b:11.0
2022-03-11 02:54:42 - sshInstance.py - [line:50] - root - INFO - 
2022-03-11 02:54:47 - test_network_sriov_stress.py - [line:171] - root - INFO - Successfuly bound VF to vfio-pci
2022-03-11 02:54:50 - sshInstance.py - [line:46] - root - INFO - /opt/APP/utility/DPDK_install/dpdk-21.02/usertools/dpdk-devbind.py -b vfio-pci 4b:11.1
2022-03-11 02:54:51 - sshInstance.py - [line:50] - root - INFO - 
2022-03-11 02:54:56 - test_network_sriov_stress.py - [line:171] - root - INFO - Successfuly bound VF to vfio-pci
2022-03-11 02:54:56 - test_network_sriov_stress.py - [line:421] - root - INFO - lspci -Dnn | grep 4b:11.1
2022-03-11 02:54:59 - sshInstance.py - [line:46] - root - INFO - lspci -Dnn | grep 4b:11.1
2022-03-11 02:54:59 - sshInstance.py - [line:50] - root - INFO - 0000:4b:11.1 Ethernet controller [0200]: Intel Corporation Ethernet Adaptive Virtual Function [8086:1889] (rev 02)

2022-03-11 02:55:04 - test_network_sriov_stress.py - [line:432] - root - INFO - Successfully get vf device id: 8086:1889
2022-03-11 02:55:07 - sshInstance.py - [line:46] - root - INFO - virsh net-info default | grep "Active" | awk '{print $2;}'
2022-03-11 02:55:07 - sshInstance.py - [line:50] - root - INFO - yes

2022-03-11 02:55:12 - test_network_sriov_stress.py - [line:197] - root - INFO - Default network is active? yes

2022-03-11 02:55:16 - sshInstance.py - [line:71] - root - INFO - virt-install --name VMtest1 --ram 4096 --cpu host --vcpuss=5 --network network=default --hostdev pci_0000_4b_11_0 --boot hd --disk path=/hhome/image/uvm1.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5900  --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
2022-03-11 02:55:19 - sshInstance.py - [line:71] - root - INFO - Domain creation completed.
[root@npx-cyp7-rh82 ~]# 
2022-03-11 02:55:22 - test_network_sriov_stress.py - [line:232] - root - INFO - Successfully created a VM: VMtest1
2022-03-11 02:55:26 - sshInstance.py - [line:71] - root - INFO - virt-install --name VMtest2 --ram 4096 --cpu host --vcpuss=5 --network network=default --hostdev pci_0000_4b_11_1 --boot hd --disk path=/hhome/image/uvm2.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5901  --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
2022-03-11 02:55:29 - sshInstance.py - [line:71] - root - INFO - Domain creation completed.
[root@npx-cyp7-rh82 ~]# 
2022-03-11 02:55:32 - test_network_sriov_stress.py - [line:232] - root - INFO - Successfully created a VM: VMtest2
2022-03-11 02:55:35 - sshInstance.py - [line:46] - root - INFO - virsh domifaddr VMtest1 | grep '192.168' | awk '{print $4;}'
2022-03-11 02:55:35 - sshInstance.py - [line:50] - root - INFO - 
2022-03-11 02:55:40 - test_network_sriov_stress.py - [line:249] - root - ERROR - not found IP of VMtest1
2022-03-11 02:56:44 - sshInstance.py - [line:46] - root - INFO - virsh domifaddr VMtest1 | grep '192.168' | awk '{print $4;}'
2022-03-11 02:56:44 - sshInstance.py - [line:50] - root - INFO - 192.168.122.73/24

2022-03-11 02:56:49 - test_network_sriov_stress.py - [line:244] - root - INFO - Successfully got VMtest1 IP: 192.168.122.73/24

2022-03-11 02:56:52 - sshInstance.py - [line:46] - root - INFO - virsh domifaddr VMtest2 | grep '192.168' | awk '{print $4;}'
2022-03-11 02:56:52 - sshInstance.py - [line:50] - root - INFO - 192.168.122.250/24

2022-03-11 02:56:57 - test_network_sriov_stress.py - [line:244] - root - INFO - Successfully got VMtest2 IP: 192.168.122.250/24

2022-03-11 02:56:57 - test_network_sriov_stress.py - [line:449] - root - INFO - ip of VM1: 192.168.122.73
2022-03-11 02:56:57 - test_network_sriov_stress.py - [line:450] - root - INFO - ip of VM1: 192.168.122.250
2022-03-11 02:57:01 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.122.73
PING 192.168.122.73 (192.168.122.73) 56(84) bytes of data.
64 bytes from 192.168.122.73: icmp_seq=1 ttl=64 time=0.152 ms
2022-03-11 02:57:02 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.73: icmp_seq=2 ttl=64 time=0.166 ms
2022-03-11 02:57:03 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.73: icmp_seq=3 ttl=64 time=0.085 ms
2022-03-11 02:57:04 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.73: icmp_seq=4 ttl=64 time=0.180 ms
2022-03-11 02:57:05 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.73: icmp_seq=5 ttl=64 time=0.100 ms

--- 192.168.122.73 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 118ms
rtt min/avg/max/mdev = 0.085/0.136/0.180/0.039 ms
[root@npx-cyp7-rh82 ~]# 
2022-03-11 02:57:14 - sshInstance.py - [line:94] - root - INFO - ssh root@192.168.122.73
The authenticity of host '192.168.122.73 (192.168.122.73)' can't be established.
ECDSA key fingerprint is SHA256:uiL9MQ8j64kizIc2NkvuxVE93C2YBfGXw5p1QHW5dUA.
Are you sure you want to continue connecting (yes/no/[fingerprint])? 
2022-03-11 02:57:23 - sshInstance.py - [line:94] - root - INFO - yes
Warning: Permanently added '192.168.122.73' (ECDSA) to the list of known hosts.
root@192.168.122.73's password: 
2022-03-11 02:57:32 - sshInstance.py - [line:94] - root - INFO - 
Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
  System information as of Thu 10 Mar 2022 01:59:03 PM EST
  System load:  0.07               Processes:             136
  Usage of /:   37.8% of 19.07GB   Users logged in:       0
  Memory usage: 4%                 IPv4 address for ens3: 192.168.122.73
  Swap usage:   0%

372 updates can be installed immediately.
227 of these updates are security updates.
To see these additional updates run: apt list --upgradable

The list of available updates is more than a week old.
To check for new updates run: sudo apt update
Last login: Thu Mar 10 05:25:32 2022 from 192.168.122.1
root@localhost:~# 
2022-03-11 02:57:41 - sshInstance.py - [line:94] - root - INFO - lspci -Dnn | grep "8086:1889" --color=never| awk '{print $1;}' [A        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [K            '
0000:00:05.0
root@localhost:~# 
2022-03-11 02:57:47 - test_network_sriov_stress.py - [line:340] - root - INFO - successfully got vf pci id of vm1: 0000:00:05.0
2022-03-11 02:57:53 - sshInstance.py - [line:94] - root - INFO - ls /sys/bus/pci/devices/0000\:00\:05.0/net/ --color=never
ens5
root@localhost:~# 
2022-03-11 02:57:58 - test_network_sriov_stress.py - [line:354] - root - INFO - Successfully got vf device name of vm1: ens5
2022-03-11 02:58:04 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 192.168.10.12
root@localhost:~# 
2022-03-11 02:58:07 - test_network_sriov_stress.py - [line:301] - root - INFO - Successfully assigned ip to ethernet: ens5
2022-03-11 02:58:13 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 down
root@localhost:~# 
2022-03-11 02:58:22 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 hw ether 1234567890ac
root@localhost:~# 
2022-03-11 02:58:31 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 up
root@localhost:~# 
2022-03-11 02:58:40 - sshInstance.py - [line:94] - root - INFO - nmcli device connect ens5
Command 'nmcli' not found, but can be installed with:
apt install network-manager
root@localhost:~# 
2022-03-11 02:58:49 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5
ens5: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.10.12  netmask 255.255.255.0  broadcast 192.168.10.255
        inet6 fe80::1034:56ff:fe78:90ac  prefixlen 64  scopeid 0x20<link>
        ether 12:34:56:78:90:ac  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 9  bytes 766 (766.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
root@localhost:~# 
2022-03-11 02:58:58 - sshInstance.py - [line:94] - root - INFO - exit
logout
Connection to 192.168.122.73 closed.
[root@npx-cyp7-rh82 ~]# 
2022-03-11 02:59:05 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.122.250
PING 192.168.122.250 (192.168.122.250) 56(84) bytes of data.
64 bytes from 192.168.122.250: icmp_seq=1 ttl=64 time=0.180 ms
2022-03-11 02:59:06 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.250: icmp_seq=2 ttl=64 time=0.179 ms
2022-03-11 02:59:07 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.250: icmp_seq=3 ttl=64 time=0.133 ms
2022-03-11 02:59:08 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.250: icmp_seq=4 ttl=64 time=0.152 ms
2022-03-11 02:59:09 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.250: icmp_seq=5 ttl=64 time=0.107 ms

--- 192.168.122.250 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 123ms
rtt min/avg/max/mdev = 0.107/0.150/0.180/0.028 ms
[root@npx-cyp7-rh82 ~]# 
2022-03-11 02:59:18 - sshInstance.py - [line:94] - root - INFO - ssh root@192.168.122.250
The authenticity of host '192.168.122.250 (192.168.122.250)' can't be established.
ECDSA key fingerprint is SHA256:uiL9MQ8j64kizIc2NkvuxVE93C2YBfGXw5p1QHW5dUA.
Are you sure you want to continue connecting (yes/no/[fingerprint])? 
2022-03-11 02:59:27 - sshInstance.py - [line:94] - root - INFO - yes
Warning: Permanently added '192.168.122.250' (ECDSA) to the list of known hosts.
root@192.168.122.250's password: 
2022-03-11 02:59:36 - sshInstance.py - [line:94] - root - INFO - 
Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
  System information as of Thu 10 Mar 2022 02:01:07 PM EST
  System load:  0.0                Processes:             136
  Usage of /:   37.8% of 19.07GB   Users logged in:       0
  Memory usage: 4%                 IPv4 address for ens3: 192.168.122.250
  Swap usage:   0%

372 updates can be installed immediately.
227 of these updates are security updates.
To see these additional updates run: apt list --upgradable

The list of available updates is more than a week old.
To check for new updates run: sudo apt update
Last login: Thu Mar 10 05:26:29 2022 from 192.168.122.1
root@localhost:~# 
2022-03-11 02:59:45 - sshInstance.py - [line:94] - root - INFO - lspci -Dnn | grep "8086:1889" --color=never| awk '{print $1;}' [A        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [K            '
0000:00:05.0
root@localhost:~# 
2022-03-11 02:59:51 - test_network_sriov_stress.py - [line:340] - root - INFO - successfully got vf pci id of vm1: 0000:00:05.0
2022-03-11 02:59:57 - sshInstance.py - [line:94] - root - INFO - ls /sys/bus/pci/devices/0000\:00\:05.0/net/ --color=never
ens5
root@localhost:~# 
2022-03-11 03:00:02 - test_network_sriov_stress.py - [line:354] - root - INFO - Successfully got vf device name of vm1: ens5
2022-03-11 03:00:08 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 192.168.10.13
root@localhost:~# 
2022-03-11 03:00:11 - test_network_sriov_stress.py - [line:301] - root - INFO - Successfully assigned ip to ethernet: ens5
2022-03-11 03:00:17 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 down
root@localhost:~# 
2022-03-11 03:00:26 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 hw ether 1234567890ab
root@localhost:~# 
2022-03-11 03:00:35 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 up
root@localhost:~# 
2022-03-11 03:00:44 - sshInstance.py - [line:94] - root - INFO - nmcli device connect ens5
Command 'nmcli' not found, but can be installed with:
apt install network-manager
root@localhost:~# 
2022-03-11 03:00:53 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5
ens5: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.10.13  netmask 255.255.255.0  broadcast 192.168.10.255
        inet6 fe80::1034:56ff:fe78:90ab  prefixlen 64  scopeid 0x20<link>
        ether 12:34:56:78:90:ab  txqueuelen 1000  (Ethernet)
        RX packets 6  bytes 2076 (2.0 KB)
        RX errors 0  dropped 2  overruns 0  frame 0
        TX packets 9  bytes 766 (766.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
root@localhost:~# 
2022-03-11 03:01:02 - sshInstance.py - [line:94] - root - INFO - exit
logout
Connection to 192.168.122.250 closed.
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:01:05 - test_network_sriov_stress.py - [line:468] - root - INFO - Ping two vms IP from SUT:
2022-03-11 03:01:09 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.10.12
PING 192.168.10.12 (192.168.10.12) 56(84) bytes of data.
64 bytes from 192.168.10.12: icmp_seq=1 ttl=64 time=4.42 ms
2022-03-11 03:01:10 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=2 ttl=64 time=0.051 ms
2022-03-11 03:01:11 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=3 ttl=64 time=0.055 ms
2022-03-11 03:01:12 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=4 ttl=64 time=0.045 ms
2022-03-11 03:01:13 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=5 ttl=64 time=0.042 ms

--- 192.168.10.12 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 65ms
rtt min/avg/max/mdev = 0.042/0.922/4.418/1.748 ms
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:01:20 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.10.13
PING 192.168.10.13 (192.168.10.13) 56(84) bytes of data.
64 bytes from 192.168.10.13: icmp_seq=1 ttl=64 time=0.205 ms
2022-03-11 03:01:21 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=2 ttl=64 time=0.047 ms
2022-03-11 03:01:22 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=3 ttl=64 time=0.043 ms
2022-03-11 03:01:23 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=4 ttl=64 time=0.033 ms
2022-03-11 03:01:24 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=5 ttl=64 time=0.041 ms

--- 192.168.10.13 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 122ms
rtt min/avg/max/mdev = 0.033/0.073/0.205/0.066 ms
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:01:27 - test_network_sriov_stress.py - [line:476] - root - INFO - ping VMtest2 IP from VMtest1
2022-03-11 03:01:31 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.122.73
PING 192.168.122.73 (192.168.122.73) 56(84) bytes of data.
64 bytes from 192.168.122.73: icmp_seq=1 ttl=64 time=0.150 ms
2022-03-11 03:01:32 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.73: icmp_seq=2 ttl=64 time=0.087 ms
2022-03-11 03:01:33 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.73: icmp_seq=3 ttl=64 time=0.085 ms
2022-03-11 03:01:34 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.73: icmp_seq=4 ttl=64 time=0.091 ms
2022-03-11 03:01:35 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.73: icmp_seq=5 ttl=64 time=0.152 ms

--- 192.168.122.73 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 113ms
rtt min/avg/max/mdev = 0.085/0.113/0.152/0.031 ms
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:01:44 - sshInstance.py - [line:94] - root - INFO - ssh root@192.168.122.73
root@192.168.122.73's password: 
2022-03-11 03:01:53 - sshInstance.py - [line:94] - root - INFO - 
Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
  System information as of Thu 10 Mar 2022 02:03:24 PM EST
  System load:  0.0                Processes:             131
  Usage of /:   37.8% of 19.07GB   Users logged in:       0
  Memory usage: 4%                 IPv4 address for ens3: 192.168.122.73
  Swap usage:   0%                 IPv4 address for ens5: 192.168.10.12

372 updates can be installed immediately.
227 of these updates are security updates.
To see these additional updates run: apt list --upgradable

The list of available updates is more than a week old.
To check for new updates run: sudo apt update
Last login: Thu Mar 10 13:59:03 2022 from 192.168.122.1
root@localhost:~# 
2022-03-11 03:02:00 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.10.13
PING 192.168.10.13 (192.168.10.13) 56(84) bytes of data.
64 bytes from 192.168.10.13: icmp_seq=1 ttl=64 time=0.146 ms
2022-03-11 03:02:01 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=2 ttl=64 time=0.099 ms
2022-03-11 03:02:02 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=3 ttl=64 time=0.055 ms
2022-03-11 03:02:03 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=4 ttl=64 time=0.058 ms
2022-03-11 03:02:04 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=5 ttl=64 time=0.052 ms

--- 192.168.10.13 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4088ms
rtt min/avg/max/mdev = 0.052/0.082/0.146/0.036 ms
root@localhost:~# 
2022-03-11 03:02:44 - sshInstance.py - [line:94] - root - INFO - exit
logout
Connection to 192.168.122.73 closed.
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:02:47 - test_network_sriov_stress.py - [line:484] - root - INFO - ping VMtest2 IP from VMtest1
2022-03-11 03:02:51 - sshInstance.py - [line:71] - root - INFO - ping -c 5 -I 192.168.10.10 192.168.10.11
PING 192.168.10.11 (192.168.10.11) from 192.168.10.10 : 56(84) bytes of data.
64 bytes from 192.168.10.11: icmp_seq=1 ttl=64 time=0.438 ms
2022-03-11 03:02:52 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=2 ttl=64 time=0.186 ms
2022-03-11 03:02:53 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=3 ttl=64 time=0.178 ms
2022-03-11 03:02:54 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=4 ttl=64 time=0.216 ms
2022-03-11 03:02:55 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=5 ttl=64 time=0.174 ms

--- 192.168.10.11 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 78ms
rtt min/avg/max/mdev = 0.174/0.238/0.438/0.101 ms
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:03:01 - sshInstance.py - [line:46] - root - INFO - echo 0 > /sys/class/net/ens785f2/device/sriov_numvfs
2022-03-11 03:03:04 - sshInstance.py - [line:50] - root - INFO - 
2022-03-11 03:03:09 - test_network_sriov_stress.py - [line:263] - root - INFO - Successfully removed VFs from PF.
2022-03-11 03:03:12 - sshInstance.py - [line:46] - root - INFO - virsh destroy VMtest1
2022-03-11 03:03:13 - sshInstance.py - [line:50] - root - INFO - Domain VMtest1 destroyed


2022-03-11 03:03:18 - test_network_sriov_stress.py - [line:276] - root - INFO - Successfully shut down the VM: VMtest1
2022-03-11 03:03:21 - sshInstance.py - [line:46] - root - INFO - virsh undefine VMtest1
2022-03-11 03:03:21 - sshInstance.py - [line:50] - root - INFO - Domain VMtest1 has been undefined


2022-03-11 03:03:26 - test_network_sriov_stress.py - [line:286] - root - INFO - Successfully removed the VM: VMtest1
2022-03-11 03:03:29 - sshInstance.py - [line:46] - root - INFO - virsh destroy VMtest2
2022-03-11 03:03:30 - sshInstance.py - [line:50] - root - INFO - Domain VMtest2 destroyed


2022-03-11 03:03:35 - test_network_sriov_stress.py - [line:276] - root - INFO - Successfully shut down the VM: VMtest2
2022-03-11 03:03:38 - sshInstance.py - [line:46] - root - INFO - virsh undefine VMtest2
2022-03-11 03:03:38 - sshInstance.py - [line:50] - root - INFO - Domain VMtest2 has been undefined


2022-03-11 03:03:43 - test_network_sriov_stress.py - [line:286] - root - INFO - Successfully removed the VM: VMtest2
2022-03-11 03:03:43 - test_network_sriov_stress.py - [line:389] - root - INFO - 

********************************************************Stress Disable and Enable for 2 time when VF is Assigned VM guest.*************************************************************


2022-03-11 03:03:46 - sshInstance.py - [line:46] - root - INFO - /opt/APP/utility/DPDK_install/dpdk-21.02/usertools/dpdk-devbind.py -s | grep ens785f2 | awk '{print $1;}'
2022-03-11 03:03:47 - sshInstance.py - [line:50] - root - INFO - 0000:4b:00.2

2022-03-11 03:03:52 - test_network_sriov_stress.py - [line:110] - root - INFO - pf_id was found: 0000:4b:00.2

2022-03-11 03:03:52 - test_network_sriov_stress.py - [line:397] - root - INFO - format_pf_id is 0000\:4b\:00.2

2022-03-11 03:03:52 - test_network_sriov_stress.py - [line:400] - root - INFO - pf_id_key: 4b
2022-03-11 03:03:58 - sshInstance.py - [line:94] - root - INFO - ifconfig ens785f2 192.168.10.10
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:04:01 - test_network_sriov_stress.py - [line:301] - root - INFO - Successfully assigned ip to ethernet: ens785f2
2022-03-11 03:04:04 - sshInstance.py - [line:46] - root - INFO - echo 4 > /sys/class/net/ens785f2/device/sriov_numvfs
2022-03-11 03:04:04 - sshInstance.py - [line:50] - root - INFO - 
2022-03-11 03:04:09 - test_network_sriov_stress.py - [line:124] - root - INFO - Successfully Virtualized PF to VFs.
2022-03-11 03:04:12 - sshInstance.py - [line:46] - root - INFO -  lspci | grep "Virtual" | awk '{print $1;}'list
2022-03-11 03:04:12 - sshInstance.py - [line:50] - root - INFO - 4b:11.0
4b:11.1
4b:11.2
4b:11.3

2022-03-11 03:04:17 - test_network_sriov_stress.py - [line:138] - root - INFO - Succssfully Gotten vf_id list:

2022-03-11 03:04:20 - sshInstance.py - [line:46] - root - INFO - modprobe vfio-pci

2022-03-11 03:04:20 - sshInstance.py - [line:50] - root - INFO - 
2022-03-11 03:04:28 - sshInstance.py - [line:46] - root - INFO - /opt/APP/utility/DPDK_install/dpdk-21.02/usertools/dpdk-devbind.py -b vfio-pci 4b:11.0
2022-03-11 03:04:29 - sshInstance.py - [line:50] - root - INFO - 
2022-03-11 03:04:34 - test_network_sriov_stress.py - [line:171] - root - INFO - Successfuly bound VF to vfio-pci
2022-03-11 03:04:37 - sshInstance.py - [line:46] - root - INFO - /opt/APP/utility/DPDK_install/dpdk-21.02/usertools/dpdk-devbind.py -b vfio-pci 4b:11.1
2022-03-11 03:04:38 - sshInstance.py - [line:50] - root - INFO - 
2022-03-11 03:04:43 - test_network_sriov_stress.py - [line:171] - root - INFO - Successfuly bound VF to vfio-pci
2022-03-11 03:04:43 - test_network_sriov_stress.py - [line:421] - root - INFO - lspci -Dnn | grep 4b:11.1
2022-03-11 03:04:46 - sshInstance.py - [line:46] - root - INFO - lspci -Dnn | grep 4b:11.1
2022-03-11 03:04:46 - sshInstance.py - [line:50] - root - INFO - 0000:4b:11.1 Ethernet controller [0200]: Intel Corporation Ethernet Adaptive Virtual Function [8086:1889] (rev 02)

2022-03-11 03:04:51 - test_network_sriov_stress.py - [line:432] - root - INFO - Successfully get vf device id: 8086:1889
2022-03-11 03:04:54 - sshInstance.py - [line:46] - root - INFO - virsh net-info default | grep "Active" | awk '{print $2;}'
2022-03-11 03:04:55 - sshInstance.py - [line:50] - root - INFO - yes

2022-03-11 03:05:00 - test_network_sriov_stress.py - [line:197] - root - INFO - Default network is active? yes

2022-03-11 03:05:04 - sshInstance.py - [line:71] - root - INFO - virt-install --name VMtest1 --ram 4096 --cpu host --vcpuss=5 --network network=default --hostdev pci_0000_4b_11_0 --boot hd --disk path=/hhome/image/uvm1.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5900  --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
ERROR    Requested operation is not valid: PCI device 0000:4b:11.0 is in use by driver QEMU, domain VMtest1
Domain installation does not appear to have been successful.
If it was, you can restart your domain by running:
  virsh --connect qemu:///system start VMtest1
otherwise, please restart your installation.
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:05:07 - test_network_sriov_stress.py - [line:229] - root - ERROR - Failed to create a VM: virt-install --name VMtest1 --ram 4096 --cpu host --vcpuss=5 --network network=default --hostdev pci_0000_4b_11_0 --boot hd --disk path=/hhome/image/uvm1.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5900  --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
ERROR    Requested operation is not valid: PCI device 0000:4b:11.0 is in use by driver QEMU, domain VMtest1
Domain installation does not appear to have been successful.
If it was, you can restart your domain by running:
  virsh --connect qemu:///system start VMtest1
otherwise, please restart your installation.
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:05:07 - test_network_sriov_stress.py - [line:230] - root - ERROR - Failed to create a VM: 
2022-03-11 03:05:11 - sshInstance.py - [line:71] - root - INFO - virt-install --name VMtest1 --ram 4096 --cpu host --vcpuss=5 --network network=default --hostdev pci_0000_4b_11_0 --boot hd --disk path=/hhome/image/uvm1.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5900  --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
2022-03-11 03:05:14 - sshInstance.py - [line:71] - root - INFO - Domain creation completed.
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:05:17 - test_network_sriov_stress.py - [line:232] - root - INFO - Successfully created a VM: VMtest1
2022-03-11 03:05:21 - sshInstance.py - [line:71] - root - INFO - virt-install --name VMtest2 --ram 4096 --cpu host --vcpuss=5 --network network=default --hostdev pci_0000_4b_11_1 --boot hd --disk path=/hhome/image/uvm2.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5901  --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
ERROR    Requested operation is not valid: PCI device 0000:4b:11.1 is in use by driver QEMU, domain VMtest2
Domain installation does not appear to have been successful.
If it was, you can restart your domain by running:
  virsh --connect qemu:///system start VMtest2
otherwise, please restart your installation.
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:05:24 - test_network_sriov_stress.py - [line:229] - root - ERROR - Failed to create a VM: virt-install --name VMtest2 --ram 4096 --cpu host --vcpuss=5 --network network=default --hostdev pci_0000_4b_11_1 --boot hd --disk path=/hhome/image/uvm2.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5901  --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
ERROR    Requested operation is not valid: PCI device 0000:4b:11.1 is in use by driver QEMU, domain VMtest2
Domain installation does not appear to have been successful.
If it was, you can restart your domain by running:
  virsh --connect qemu:///system start VMtest2
otherwise, please restart your installation.
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:05:24 - test_network_sriov_stress.py - [line:230] - root - ERROR - Failed to create a VM: 
2022-03-11 03:05:28 - sshInstance.py - [line:71] - root - INFO - virt-install --name VMtest2 --ram 4096 --cpu host --vcpuss=5 --network network=default --hostdev pci_0000_4b_11_1 --boot hd --disk path=/hhome/image/uvm2.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5901  --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
2022-03-11 03:05:31 - sshInstance.py - [line:71] - root - INFO - Domain creation completed.
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:05:34 - test_network_sriov_stress.py - [line:232] - root - INFO - Successfully created a VM: VMtest2
2022-03-11 03:05:37 - sshInstance.py - [line:46] - root - INFO - virsh domifaddr VMtest1 | grep '192.168' | awk '{print $4;}'
2022-03-11 03:05:37 - sshInstance.py - [line:50] - root - INFO - 192.168.122.73/24

2022-03-11 03:05:42 - test_network_sriov_stress.py - [line:244] - root - INFO - Successfully got VMtest1 IP: 192.168.122.73/24

2022-03-11 03:05:45 - sshInstance.py - [line:46] - root - INFO - virsh domifaddr VMtest2 | grep '192.168' | awk '{print $4;}'
2022-03-11 03:05:45 - sshInstance.py - [line:50] - root - INFO - 
2022-03-11 03:05:50 - test_network_sriov_stress.py - [line:249] - root - ERROR - not found IP of VMtest2
2022-03-11 03:06:53 - sshInstance.py - [line:46] - root - INFO - virsh domifaddr VMtest2 | grep '192.168' | awk '{print $4;}'
2022-03-11 03:06:53 - sshInstance.py - [line:50] - root - INFO - 192.168.122.250/24

2022-03-11 03:06:58 - test_network_sriov_stress.py - [line:244] - root - INFO - Successfully got VMtest2 IP: 192.168.122.250/24

2022-03-11 03:06:58 - test_network_sriov_stress.py - [line:449] - root - INFO - ip of VM1: 192.168.122.73
2022-03-11 03:06:58 - test_network_sriov_stress.py - [line:450] - root - INFO - ip of VM1: 192.168.122.250
2022-03-11 03:07:02 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.122.73
PING 192.168.122.73 (192.168.122.73) 56(84) bytes of data.
64 bytes from 192.168.122.73: icmp_seq=1 ttl=64 time=0.141 ms
2022-03-11 03:07:03 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.73: icmp_seq=2 ttl=64 time=0.124 ms
2022-03-11 03:07:04 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.73: icmp_seq=3 ttl=64 time=0.090 ms
2022-03-11 03:07:05 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.73: icmp_seq=4 ttl=64 time=0.089 ms
2022-03-11 03:07:06 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.73: icmp_seq=5 ttl=64 time=0.099 ms

--- 192.168.122.73 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 136ms
rtt min/avg/max/mdev = 0.089/0.108/0.141/0.023 ms
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:07:15 - sshInstance.py - [line:94] - root - INFO - ssh root@192.168.122.73
root@192.168.122.73's password: 
2022-03-11 03:07:24 - sshInstance.py - [line:94] - root - INFO - 
Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
  System information as of Thu 10 Mar 2022 02:08:55 PM EST
  System load:  0.03               Processes:             137
  Usage of /:   37.8% of 19.07GB   Users logged in:       0
  Memory usage: 4%                 IPv4 address for ens3: 192.168.122.73
  Swap usage:   0%

372 updates can be installed immediately.
227 of these updates are security updates.
To see these additional updates run: apt list --upgradable

The list of available updates is more than a week old.
To check for new updates run: sudo apt update
Last login: Thu Mar 10 14:03:24 2022 from 192.168.122.1
root@localhost:~# 
2022-03-11 03:07:33 - sshInstance.py - [line:94] - root - INFO - lspci -Dnn | grep "8086:1889" --color=never| awk '{print $1;}' [A        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [K            '
0000:00:05.0
root@localhost:~# 
2022-03-11 03:07:39 - test_network_sriov_stress.py - [line:340] - root - INFO - successfully got vf pci id of vm1: 0000:00:05.0
2022-03-11 03:07:45 - sshInstance.py - [line:94] - root - INFO - ls /sys/bus/pci/devices/0000\:00\:05.0/net/ --color=never
ens5
root@localhost:~# 
2022-03-11 03:07:50 - test_network_sriov_stress.py - [line:354] - root - INFO - Successfully got vf device name of vm1: ens5
2022-03-11 03:07:56 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 192.168.10.12
root@localhost:~# 
2022-03-11 03:07:59 - test_network_sriov_stress.py - [line:301] - root - INFO - Successfully assigned ip to ethernet: ens5
2022-03-11 03:08:05 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 down
root@localhost:~# 
2022-03-11 03:08:14 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 hw ether 1234567890ac
root@localhost:~# 
2022-03-11 03:08:23 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 up
root@localhost:~# 
2022-03-11 03:08:32 - sshInstance.py - [line:94] - root - INFO - nmcli device connect ens5
Command 'nmcli' not found, but can be installed with:
apt install network-manager
root@localhost:~# 
2022-03-11 03:08:41 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5
ens5: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.10.12  netmask 255.255.255.0  broadcast 192.168.10.255
        inet6 fe80::1034:56ff:fe78:90ac  prefixlen 64  scopeid 0x20<link>
        ether 12:34:56:78:90:ac  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 9  bytes 766 (766.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
root@localhost:~# 
2022-03-11 03:08:50 - sshInstance.py - [line:94] - root - INFO - exit
logout
Connection to 192.168.122.73 closed.
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:08:57 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.122.250
PING 192.168.122.250 (192.168.122.250) 56(84) bytes of data.
64 bytes from 192.168.122.250: icmp_seq=1 ttl=64 time=0.140 ms
2022-03-11 03:08:58 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.250: icmp_seq=2 ttl=64 time=0.078 ms
2022-03-11 03:08:59 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.250: icmp_seq=3 ttl=64 time=0.071 ms
2022-03-11 03:09:00 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.250: icmp_seq=4 ttl=64 time=0.073 ms
2022-03-11 03:09:01 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.250: icmp_seq=5 ttl=64 time=0.073 ms

--- 192.168.122.250 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 130ms
rtt min/avg/max/mdev = 0.071/0.087/0.140/0.026 ms
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:09:10 - sshInstance.py - [line:94] - root - INFO - ssh root@192.168.122.250
root@192.168.122.250's password: 
2022-03-11 03:09:19 - sshInstance.py - [line:94] - root - INFO - 
Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
  System information as of Thu 10 Mar 2022 02:10:50 PM EST
  System load:  0.0                Processes:             138
  Usage of /:   37.8% of 19.07GB   Users logged in:       0
  Memory usage: 4%                 IPv4 address for ens3: 192.168.122.250
  Swap usage:   0%

372 updates can be installed immediately.
227 of these updates are security updates.
To see these additional updates run: apt list --upgradable

The list of available updates is more than a week old.
To check for new updates run: sudo apt update
Last login: Thu Mar 10 14:01:08 2022 from 192.168.122.1
root@localhost:~# 
2022-03-11 03:09:28 - sshInstance.py - [line:94] - root - INFO - lspci -Dnn | grep "8086:1889" --color=never| awk '{print $1;}' [A        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [K            '
0000:00:05.0
root@localhost:~# 
2022-03-11 03:09:34 - test_network_sriov_stress.py - [line:340] - root - INFO - successfully got vf pci id of vm1: 0000:00:05.0
2022-03-11 03:09:40 - sshInstance.py - [line:94] - root - INFO - ls /sys/bus/pci/devices/0000\:00\:05.0/net/ --color=never
ens5
root@localhost:~# 
2022-03-11 03:09:45 - test_network_sriov_stress.py - [line:354] - root - INFO - Successfully got vf device name of vm1: ens5
2022-03-11 03:09:51 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 192.168.10.13
root@localhost:~# 
2022-03-11 03:09:54 - test_network_sriov_stress.py - [line:301] - root - INFO - Successfully assigned ip to ethernet: ens5
2022-03-11 03:10:00 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 down
root@localhost:~# 
2022-03-11 03:10:09 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 hw ether 1234567890ab
root@localhost:~# 
2022-03-11 03:10:18 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 up
root@localhost:~# 
2022-03-11 03:10:27 - sshInstance.py - [line:94] - root - INFO - nmcli device connect ens5
Command 'nmcli' not found, but can be installed with:
apt install network-manager
root@localhost:~# 
2022-03-11 03:10:36 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5
ens5: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.10.13  netmask 255.255.255.0  broadcast 192.168.10.255
        inet6 fe80::1034:56ff:fe78:90ab  prefixlen 64  scopeid 0x20<link>
        ether 12:34:56:78:90:ab  txqueuelen 1000  (Ethernet)
        RX packets 2  bytes 692 (692.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 9  bytes 766 (766.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
root@localhost:~# 
2022-03-11 03:10:45 - sshInstance.py - [line:94] - root - INFO - exit
logout
Connection to 192.168.122.250 closed.
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:10:48 - test_network_sriov_stress.py - [line:468] - root - INFO - Ping two vms IP from SUT:
2022-03-11 03:10:52 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.10.12
PING 192.168.10.12 (192.168.10.12) 56(84) bytes of data.
64 bytes from 192.168.10.12: icmp_seq=1 ttl=64 time=0.178 ms
2022-03-11 03:10:53 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=2 ttl=64 time=0.074 ms
2022-03-11 03:10:54 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=3 ttl=64 time=0.038 ms
2022-03-11 03:10:55 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=4 ttl=64 time=0.042 ms
2022-03-11 03:10:56 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=5 ttl=64 time=0.045 ms

--- 192.168.10.12 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 125ms
rtt min/avg/max/mdev = 0.038/0.075/0.178/0.053 ms
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:11:03 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.10.13
PING 192.168.10.13 (192.168.10.13) 56(84) bytes of data.
64 bytes from 192.168.10.13: icmp_seq=1 ttl=64 time=0.165 ms
2022-03-11 03:11:04 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=2 ttl=64 time=0.042 ms
2022-03-11 03:11:05 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=3 ttl=64 time=0.035 ms
2022-03-11 03:11:06 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=4 ttl=64 time=0.056 ms
2022-03-11 03:11:07 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=5 ttl=64 time=0.055 ms

--- 192.168.10.13 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 118ms
rtt min/avg/max/mdev = 0.035/0.070/0.165/0.048 ms
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:11:10 - test_network_sriov_stress.py - [line:476] - root - INFO - ping VMtest2 IP from VMtest1
2022-03-11 03:11:14 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.122.73
PING 192.168.122.73 (192.168.122.73) 56(84) bytes of data.
64 bytes from 192.168.122.73: icmp_seq=1 ttl=64 time=0.160 ms
2022-03-11 03:11:15 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.73: icmp_seq=2 ttl=64 time=0.145 ms
2022-03-11 03:11:16 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.73: icmp_seq=3 ttl=64 time=0.145 ms
2022-03-11 03:11:17 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.73: icmp_seq=4 ttl=64 time=0.136 ms
2022-03-11 03:11:18 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.73: icmp_seq=5 ttl=64 time=0.104 ms

--- 192.168.122.73 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 111ms
rtt min/avg/max/mdev = 0.104/0.138/0.160/0.018 ms
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:11:27 - sshInstance.py - [line:94] - root - INFO - ssh root@192.168.122.73
root@192.168.122.73's password: 
2022-03-11 03:11:36 - sshInstance.py - [line:94] - root - INFO - 
Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
  System information as of Thu 10 Mar 2022 02:13:07 PM EST
  System load:  0.0                Processes:             129
  Usage of /:   37.8% of 19.07GB   Users logged in:       0
  Memory usage: 4%                 IPv4 address for ens3: 192.168.122.73
  Swap usage:   0%                 IPv4 address for ens5: 192.168.10.12

372 updates can be installed immediately.
227 of these updates are security updates.
To see these additional updates run: apt list --upgradable

The list of available updates is more than a week old.
To check for new updates run: sudo apt update
Last login: Thu Mar 10 14:08:55 2022 from 192.168.122.1
root@localhost:~# 
2022-03-11 03:11:43 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.10.13
PING 192.168.10.13 (192.168.10.13) 56(84) bytes of data.
64 bytes from 192.168.10.13: icmp_seq=1 ttl=64 time=0.162 ms
2022-03-11 03:11:44 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=2 ttl=64 time=0.073 ms
2022-03-11 03:11:45 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=3 ttl=64 time=0.060 ms
2022-03-11 03:11:46 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=4 ttl=64 time=0.065 ms
2022-03-11 03:11:47 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=5 ttl=64 time=0.066 ms

--- 192.168.10.13 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4104ms
rtt min/avg/max/mdev = 0.060/0.085/0.162/0.038 ms
root@localhost:~# 
2022-03-11 03:12:27 - sshInstance.py - [line:94] - root - INFO - exit
logout
Connection to 192.168.122.73 closed.
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:12:30 - test_network_sriov_stress.py - [line:484] - root - INFO - ping VMtest2 IP from VMtest1
2022-03-11 03:12:34 - sshInstance.py - [line:71] - root - INFO - ping -c 5 -I 192.168.10.10 192.168.10.11
PING 192.168.10.11 (192.168.10.11) from 192.168.10.10 : 56(84) bytes of data.
64 bytes from 192.168.10.11: icmp_seq=1 ttl=64 time=0.260 ms
2022-03-11 03:12:35 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=2 ttl=64 time=0.191 ms
2022-03-11 03:12:36 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=3 ttl=64 time=0.176 ms
2022-03-11 03:12:37 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=4 ttl=64 time=0.168 ms
2022-03-11 03:12:38 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=5 ttl=64 time=0.182 ms

--- 192.168.10.11 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 140ms
rtt min/avg/max/mdev = 0.168/0.195/0.260/0.035 ms
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:12:44 - sshInstance.py - [line:46] - root - INFO - echo 0 > /sys/class/net/ens785f2/device/sriov_numvfs
2022-03-11 03:12:47 - sshInstance.py - [line:50] - root - INFO - 
2022-03-11 03:12:52 - test_network_sriov_stress.py - [line:263] - root - INFO - Successfully removed VFs from PF.
2022-03-11 03:12:55 - sshInstance.py - [line:46] - root - INFO - virsh destroy VMtest1
2022-03-11 03:12:56 - sshInstance.py - [line:50] - root - INFO - Domain VMtest1 destroyed


2022-03-11 03:13:01 - test_network_sriov_stress.py - [line:276] - root - INFO - Successfully shut down the VM: VMtest1
2022-03-11 03:13:04 - sshInstance.py - [line:46] - root - INFO - virsh undefine VMtest1
2022-03-11 03:13:04 - sshInstance.py - [line:50] - root - INFO - Domain VMtest1 has been undefined


2022-03-11 03:13:09 - test_network_sriov_stress.py - [line:286] - root - INFO - Successfully removed the VM: VMtest1
2022-03-11 03:13:12 - sshInstance.py - [line:46] - root - INFO - virsh destroy VMtest2
2022-03-11 03:13:13 - sshInstance.py - [line:50] - root - INFO - Domain VMtest2 destroyed


2022-03-11 03:13:18 - test_network_sriov_stress.py - [line:276] - root - INFO - Successfully shut down the VM: VMtest2
2022-03-11 03:13:21 - sshInstance.py - [line:46] - root - INFO - virsh undefine VMtest2
2022-03-11 03:13:21 - sshInstance.py - [line:50] - root - INFO - Domain VMtest2 has been undefined


2022-03-11 03:13:26 - test_network_sriov_stress.py - [line:286] - root - INFO - Successfully removed the VM: VMtest2
2022-03-11 03:13:26 - test_network_sriov_stress.py - [line:389] - root - INFO - 

********************************************************Stress Disable and Enable for 3 time when VF is Assigned VM guest.*************************************************************


2022-03-11 03:13:29 - sshInstance.py - [line:46] - root - INFO - /opt/APP/utility/DPDK_install/dpdk-21.02/usertools/dpdk-devbind.py -s | grep ens785f2 | awk '{print $1;}'
2022-03-11 03:13:30 - sshInstance.py - [line:50] - root - INFO - 0000:4b:00.2

2022-03-11 03:13:35 - test_network_sriov_stress.py - [line:110] - root - INFO - pf_id was found: 0000:4b:00.2

2022-03-11 03:13:35 - test_network_sriov_stress.py - [line:397] - root - INFO - format_pf_id is 0000\:4b\:00.2

2022-03-11 03:13:35 - test_network_sriov_stress.py - [line:400] - root - INFO - pf_id_key: 4b
2022-03-11 03:13:41 - sshInstance.py - [line:94] - root - INFO - ifconfig ens785f2 192.168.10.10
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:13:44 - test_network_sriov_stress.py - [line:301] - root - INFO - Successfully assigned ip to ethernet: ens785f2
2022-03-11 03:13:47 - sshInstance.py - [line:46] - root - INFO - echo 4 > /sys/class/net/ens785f2/device/sriov_numvfs
2022-03-11 03:13:47 - sshInstance.py - [line:50] - root - INFO - 
2022-03-11 03:13:52 - test_network_sriov_stress.py - [line:124] - root - INFO - Successfully Virtualized PF to VFs.
2022-03-11 03:13:55 - sshInstance.py - [line:46] - root - INFO -  lspci | grep "Virtual" | awk '{print $1;}'list
2022-03-11 03:13:55 - sshInstance.py - [line:50] - root - INFO - 4b:11.0
4b:11.1
4b:11.2
4b:11.3

2022-03-11 03:14:00 - test_network_sriov_stress.py - [line:138] - root - INFO - Succssfully Gotten vf_id list:

2022-03-11 03:14:03 - sshInstance.py - [line:46] - root - INFO - modprobe vfio-pci

2022-03-11 03:14:03 - sshInstance.py - [line:50] - root - INFO - 
2022-03-11 03:14:11 - sshInstance.py - [line:46] - root - INFO - /opt/APP/utility/DPDK_install/dpdk-21.02/usertools/dpdk-devbind.py -b vfio-pci 4b:11.0
2022-03-11 03:14:12 - sshInstance.py - [line:50] - root - INFO - 
2022-03-11 03:14:17 - test_network_sriov_stress.py - [line:171] - root - INFO - Successfuly bound VF to vfio-pci
2022-03-11 03:14:20 - sshInstance.py - [line:46] - root - INFO - /opt/APP/utility/DPDK_install/dpdk-21.02/usertools/dpdk-devbind.py -b vfio-pci 4b:11.1
2022-03-11 03:14:22 - sshInstance.py - [line:50] - root - INFO - 
2022-03-11 03:14:27 - test_network_sriov_stress.py - [line:171] - root - INFO - Successfuly bound VF to vfio-pci
2022-03-11 03:14:27 - test_network_sriov_stress.py - [line:421] - root - INFO - lspci -Dnn | grep 4b:11.1
2022-03-11 03:14:30 - sshInstance.py - [line:46] - root - INFO - lspci -Dnn | grep 4b:11.1
2022-03-11 03:14:30 - sshInstance.py - [line:50] - root - INFO - 0000:4b:11.1 Ethernet controller [0200]: Intel Corporation Ethernet Adaptive Virtual Function [8086:1889] (rev 02)

2022-03-11 03:14:35 - test_network_sriov_stress.py - [line:432] - root - INFO - Successfully get vf device id: 8086:1889
2022-03-11 03:14:38 - sshInstance.py - [line:46] - root - INFO - virsh net-info default | grep "Active" | awk '{print $2;}'
2022-03-11 03:14:38 - sshInstance.py - [line:50] - root - INFO - yes

2022-03-11 03:14:43 - test_network_sriov_stress.py - [line:197] - root - INFO - Default network is active? yes

2022-03-11 03:14:47 - sshInstance.py - [line:71] - root - INFO - virt-install --name VMtest1 --ram 4096 --cpu host --vcpuss=5 --network network=default --hostdev pci_0000_4b_11_0 --boot hd --disk path=/hhome/image/uvm1.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5900  --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
ERROR    Requested operation is not valid: PCI device 0000:4b:11.0 is in use by driver QEMU, domain VMtest1
Domain installation does not appear to have been successful.
If it was, you can restart your domain by running:
  virsh --connect qemu:///system start VMtest1
otherwise, please restart your installation.
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:14:50 - test_network_sriov_stress.py - [line:229] - root - ERROR - Failed to create a VM: virt-install --name VMtest1 --ram 4096 --cpu host --vcpuss=5 --network network=default --hostdev pci_0000_4b_11_0 --boot hd --disk path=/hhome/image/uvm1.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5900  --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
ERROR    Requested operation is not valid: PCI device 0000:4b:11.0 is in use by driver QEMU, domain VMtest1
Domain installation does not appear to have been successful.
If it was, you can restart your domain by running:
  virsh --connect qemu:///system start VMtest1
otherwise, please restart your installation.
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:14:50 - test_network_sriov_stress.py - [line:230] - root - ERROR - Failed to create a VM: 
2022-03-11 03:14:54 - sshInstance.py - [line:71] - root - INFO - virt-install --name VMtest1 --ram 4096 --cpu host --vcpuss=5 --network network=default --hostdev pci_0000_4b_11_0 --boot hd --disk path=/hhome/image/uvm1.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5900  --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
2022-03-11 03:14:57 - sshInstance.py - [line:71] - root - INFO - Domain creation completed.
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:15:00 - test_network_sriov_stress.py - [line:232] - root - INFO - Successfully created a VM: VMtest1
2022-03-11 03:15:04 - sshInstance.py - [line:71] - root - INFO - virt-install --name VMtest2 --ram 4096 --cpu host --vcpuss=5 --network network=default --hostdev pci_0000_4b_11_1 --boot hd --disk path=/hhome/image/uvm2.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5901  --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
ERROR    Requested operation is not valid: PCI device 0000:4b:11.1 is in use by driver QEMU, domain VMtest2
Domain installation does not appear to have been successful.
If it was, you can restart your domain by running:
  virsh --connect qemu:///system start VMtest2
otherwise, please restart your installation.
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:15:07 - test_network_sriov_stress.py - [line:229] - root - ERROR - Failed to create a VM: virt-install --name VMtest2 --ram 4096 --cpu host --vcpuss=5 --network network=default --hostdev pci_0000_4b_11_1 --boot hd --disk path=/hhome/image/uvm2.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5901  --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
ERROR    Requested operation is not valid: PCI device 0000:4b:11.1 is in use by driver QEMU, domain VMtest2
Domain installation does not appear to have been successful.
If it was, you can restart your domain by running:
  virsh --connect qemu:///system start VMtest2
otherwise, please restart your installation.
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:15:07 - test_network_sriov_stress.py - [line:230] - root - ERROR - Failed to create a VM: 
2022-03-11 03:15:11 - sshInstance.py - [line:71] - root - INFO - virt-install --name VMtest2 --ram 4096 --cpu host --vcpuss=5 --network network=default --hostdev pci_0000_4b_11_1 --boot hd --disk path=/hhome/image/uvm2.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5901  --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
2022-03-11 03:15:14 - sshInstance.py - [line:71] - root - INFO - Domain creation completed.
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:15:17 - test_network_sriov_stress.py - [line:232] - root - INFO - Successfully created a VM: VMtest2
2022-03-11 03:15:20 - sshInstance.py - [line:46] - root - INFO - virsh domifaddr VMtest1 | grep '192.168' | awk '{print $4;}'
2022-03-11 03:15:20 - sshInstance.py - [line:50] - root - INFO - 192.168.122.73/24

2022-03-11 03:15:25 - test_network_sriov_stress.py - [line:244] - root - INFO - Successfully got VMtest1 IP: 192.168.122.73/24

2022-03-11 03:15:28 - sshInstance.py - [line:46] - root - INFO - virsh domifaddr VMtest2 | grep '192.168' | awk '{print $4;}'
2022-03-11 03:15:28 - sshInstance.py - [line:50] - root - INFO - 
2022-03-11 03:15:33 - test_network_sriov_stress.py - [line:249] - root - ERROR - not found IP of VMtest2
2022-03-11 03:16:36 - sshInstance.py - [line:46] - root - INFO - virsh domifaddr VMtest2 | grep '192.168' | awk '{print $4;}'
2022-03-11 03:16:36 - sshInstance.py - [line:50] - root - INFO - 192.168.122.250/24

2022-03-11 03:16:41 - test_network_sriov_stress.py - [line:244] - root - INFO - Successfully got VMtest2 IP: 192.168.122.250/24

2022-03-11 03:16:41 - test_network_sriov_stress.py - [line:449] - root - INFO - ip of VM1: 192.168.122.73
2022-03-11 03:16:41 - test_network_sriov_stress.py - [line:450] - root - INFO - ip of VM1: 192.168.122.250
2022-03-11 03:16:45 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.122.73
PING 192.168.122.73 (192.168.122.73) 56(84) bytes of data.
64 bytes from 192.168.122.73: icmp_seq=1 ttl=64 time=0.151 ms
2022-03-11 03:16:46 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.73: icmp_seq=2 ttl=64 time=0.115 ms
2022-03-11 03:16:47 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.73: icmp_seq=3 ttl=64 time=0.084 ms
2022-03-11 03:16:48 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.73: icmp_seq=4 ttl=64 time=0.137 ms
2022-03-11 03:16:49 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.73: icmp_seq=5 ttl=64 time=0.083 ms

--- 192.168.122.73 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 86ms
rtt min/avg/max/mdev = 0.083/0.114/0.151/0.027 ms
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:16:58 - sshInstance.py - [line:94] - root - INFO - ssh root@192.168.122.73
root@192.168.122.73's password: 
2022-03-11 03:17:07 - sshInstance.py - [line:94] - root - INFO - 
Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
  System information as of Thu 10 Mar 2022 02:18:38 PM EST
  System load:  0.03               Processes:             138
  Usage of /:   37.9% of 19.07GB   Users logged in:       0
  Memory usage: 4%                 IPv4 address for ens3: 192.168.122.73
  Swap usage:   0%

372 updates can be installed immediately.
227 of these updates are security updates.
To see these additional updates run: apt list --upgradable

The list of available updates is more than a week old.
To check for new updates run: sudo apt update
Last login: Thu Mar 10 14:13:07 2022 from 192.168.122.1
root@localhost:~# 
2022-03-11 03:17:16 - sshInstance.py - [line:94] - root - INFO - lspci -Dnn | grep "8086:1889" --color=never| awk '{print $1;}' [A        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [K            '
0000:00:05.0
root@localhost:~# 
2022-03-11 03:17:22 - test_network_sriov_stress.py - [line:340] - root - INFO - successfully got vf pci id of vm1: 0000:00:05.0
2022-03-11 03:17:28 - sshInstance.py - [line:94] - root - INFO - ls /sys/bus/pci/devices/0000\:00\:05.0/net/ --color=never
ens5
root@localhost:~# 
2022-03-11 03:17:33 - test_network_sriov_stress.py - [line:354] - root - INFO - Successfully got vf device name of vm1: ens5
2022-03-11 03:17:39 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 192.168.10.12
root@localhost:~# 
2022-03-11 03:17:42 - test_network_sriov_stress.py - [line:301] - root - INFO - Successfully assigned ip to ethernet: ens5
2022-03-11 03:17:48 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 down
root@localhost:~# 
2022-03-11 03:17:57 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 hw ether 1234567890ac
root@localhost:~# 
2022-03-11 03:18:06 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 up
root@localhost:~# 
2022-03-11 03:18:15 - sshInstance.py - [line:94] - root - INFO - nmcli device connect ens5
Command 'nmcli' not found, but can be installed with:
apt install network-manager
root@localhost:~# 
2022-03-11 03:18:24 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5
ens5: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.10.12  netmask 255.255.255.0  broadcast 192.168.10.255
        inet6 fe80::1034:56ff:fe78:90ac  prefixlen 64  scopeid 0x20<link>
        ether 12:34:56:78:90:ac  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 9  bytes 766 (766.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
root@localhost:~# 
2022-03-11 03:18:33 - sshInstance.py - [line:94] - root - INFO - exit
logout
Connection to 192.168.122.73 closed.
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:18:40 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.122.250
PING 192.168.122.250 (192.168.122.250) 56(84) bytes of data.
64 bytes from 192.168.122.250: icmp_seq=1 ttl=64 time=0.150 ms
2022-03-11 03:18:41 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.250: icmp_seq=2 ttl=64 time=0.088 ms
2022-03-11 03:18:42 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.250: icmp_seq=3 ttl=64 time=0.092 ms
2022-03-11 03:18:43 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.250: icmp_seq=4 ttl=64 time=0.076 ms
2022-03-11 03:18:44 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.250: icmp_seq=5 ttl=64 time=0.083 ms

--- 192.168.122.250 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 82ms
rtt min/avg/max/mdev = 0.076/0.097/0.150/0.029 ms
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:18:53 - sshInstance.py - [line:94] - root - INFO - ssh root@192.168.122.250
root@192.168.122.250's password: 
2022-03-11 03:19:02 - sshInstance.py - [line:94] - root - INFO - 
Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
  System information as of Thu 10 Mar 2022 02:20:33 PM EST
  System load:  0.01               Processes:             136
  Usage of /:   37.8% of 19.07GB   Users logged in:       0
  Memory usage: 4%                 IPv4 address for ens3: 192.168.122.250
  Swap usage:   0%

372 updates can be installed immediately.
227 of these updates are security updates.
To see these additional updates run: apt list --upgradable

The list of available updates is more than a week old.
To check for new updates run: sudo apt update
Last login: Thu Mar 10 14:10:50 2022 from 192.168.122.1
root@localhost:~# 
2022-03-11 03:19:11 - sshInstance.py - [line:94] - root - INFO - lspci -Dnn | grep "8086:1889" --color=never| awk '{print $1;}' [A        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [K            '
0000:00:05.0
root@localhost:~# 
2022-03-11 03:19:17 - test_network_sriov_stress.py - [line:340] - root - INFO - successfully got vf pci id of vm1: 0000:00:05.0
2022-03-11 03:19:23 - sshInstance.py - [line:94] - root - INFO - ls /sys/bus/pci/devices/0000\:00\:05.0/net/ --color=never
ens5
root@localhost:~# 
2022-03-11 03:19:28 - test_network_sriov_stress.py - [line:354] - root - INFO - Successfully got vf device name of vm1: ens5
2022-03-11 03:19:34 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 192.168.10.13
root@localhost:~# 
2022-03-11 03:19:37 - test_network_sriov_stress.py - [line:301] - root - INFO - Successfully assigned ip to ethernet: ens5
2022-03-11 03:19:43 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 down
root@localhost:~# 
2022-03-11 03:19:52 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 hw ether 1234567890ab
root@localhost:~# 
2022-03-11 03:20:01 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 up
root@localhost:~# 
2022-03-11 03:20:10 - sshInstance.py - [line:94] - root - INFO - nmcli device connect ens5
Command 'nmcli' not found, but can be installed with:
apt install network-manager
root@localhost:~# 
2022-03-11 03:20:19 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5
ens5: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.10.13  netmask 255.255.255.0  broadcast 192.168.10.255
        inet6 fe80::1034:56ff:fe78:90ab  prefixlen 64  scopeid 0x20<link>
        ether 12:34:56:78:90:ab  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 9  bytes 766 (766.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
root@localhost:~# 
2022-03-11 03:20:28 - sshInstance.py - [line:94] - root - INFO - exit
logout
Connection to 192.168.122.250 closed.
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:20:31 - test_network_sriov_stress.py - [line:468] - root - INFO - Ping two vms IP from SUT:
2022-03-11 03:20:35 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.10.12
PING 192.168.10.12 (192.168.10.12) 56(84) bytes of data.
64 bytes from 192.168.10.12: icmp_seq=1 ttl=64 time=0.195 ms
2022-03-11 03:20:36 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=2 ttl=64 time=0.045 ms
2022-03-11 03:20:37 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=3 ttl=64 time=0.061 ms
2022-03-11 03:20:38 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=4 ttl=64 time=0.045 ms
2022-03-11 03:20:39 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=5 ttl=64 time=0.040 ms

--- 192.168.10.12 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 78ms
rtt min/avg/max/mdev = 0.040/0.077/0.195/0.059 ms
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:20:46 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.10.13
PING 192.168.10.13 (192.168.10.13) 56(84) bytes of data.
64 bytes from 192.168.10.13: icmp_seq=1 ttl=64 time=0.187 ms
2022-03-11 03:20:47 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=2 ttl=64 time=0.058 ms
2022-03-11 03:20:48 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=3 ttl=64 time=0.035 ms
2022-03-11 03:20:49 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=4 ttl=64 time=0.037 ms
2022-03-11 03:20:50 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=5 ttl=64 time=0.049 ms

--- 192.168.10.13 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 135ms
rtt min/avg/max/mdev = 0.035/0.073/0.187/0.057 ms
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:20:53 - test_network_sriov_stress.py - [line:476] - root - INFO - ping VMtest2 IP from VMtest1
2022-03-11 03:20:57 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.122.73
PING 192.168.122.73 (192.168.122.73) 56(84) bytes of data.
64 bytes from 192.168.122.73: icmp_seq=1 ttl=64 time=0.143 ms
2022-03-11 03:20:58 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.73: icmp_seq=2 ttl=64 time=0.133 ms
2022-03-11 03:20:59 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.73: icmp_seq=3 ttl=64 time=0.100 ms
2022-03-11 03:21:00 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.73: icmp_seq=4 ttl=64 time=0.087 ms
2022-03-11 03:21:01 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.73: icmp_seq=5 ttl=64 time=0.119 ms

--- 192.168.122.73 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 127ms
rtt min/avg/max/mdev = 0.087/0.116/0.143/0.022 ms
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:21:10 - sshInstance.py - [line:94] - root - INFO - ssh root@192.168.122.73
root@192.168.122.73's password: 
2022-03-11 03:21:19 - sshInstance.py - [line:94] - root - INFO - 
Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
  System information as of Thu 10 Mar 2022 02:22:51 PM EST
  System load:  0.0                Processes:             127
  Usage of /:   37.9% of 19.07GB   Users logged in:       0
  Memory usage: 4%                 IPv4 address for ens3: 192.168.122.73
  Swap usage:   0%                 IPv4 address for ens5: 192.168.10.12

372 updates can be installed immediately.
227 of these updates are security updates.
To see these additional updates run: apt list --upgradable

The list of available updates is more than a week old.
To check for new updates run: sudo apt update
Last login: Thu Mar 10 14:18:39 2022 from 192.168.122.1
root@localhost:~# 
2022-03-11 03:21:26 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.10.13
PING 192.168.10.13 (192.168.10.13) 56(84) bytes of data.
64 bytes from 192.168.10.13: icmp_seq=1 ttl=64 time=0.200 ms
2022-03-11 03:21:27 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=2 ttl=64 time=0.062 ms
2022-03-11 03:21:28 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=3 ttl=64 time=0.051 ms
2022-03-11 03:21:29 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=4 ttl=64 time=0.042 ms
2022-03-11 03:21:30 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=5 ttl=64 time=0.070 ms

--- 192.168.10.13 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4091ms
rtt min/avg/max/mdev = 0.042/0.085/0.200/0.058 ms
root@localhost:~# 
2022-03-11 03:22:10 - sshInstance.py - [line:94] - root - INFO - exit
logout
Connection to 192.168.122.73 closed.
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:22:13 - test_network_sriov_stress.py - [line:484] - root - INFO - ping VMtest2 IP from VMtest1
2022-03-11 03:22:17 - sshInstance.py - [line:71] - root - INFO - ping -c 5 -I 192.168.10.10 192.168.10.11
PING 192.168.10.11 (192.168.10.11) from 192.168.10.10 : 56(84) bytes of data.
64 bytes from 192.168.10.11: icmp_seq=1 ttl=64 time=0.253 ms
2022-03-11 03:22:18 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=2 ttl=64 time=0.179 ms
2022-03-11 03:22:19 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=3 ttl=64 time=0.186 ms
2022-03-11 03:22:20 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=4 ttl=64 time=0.117 ms
2022-03-11 03:22:21 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=5 ttl=64 time=0.185 ms

--- 192.168.10.11 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 92ms
rtt min/avg/max/mdev = 0.117/0.184/0.253/0.043 ms
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:22:27 - sshInstance.py - [line:46] - root - INFO - echo 0 > /sys/class/net/ens785f2/device/sriov_numvfs
2022-03-11 03:22:30 - sshInstance.py - [line:50] - root - INFO - 
2022-03-11 03:22:35 - test_network_sriov_stress.py - [line:263] - root - INFO - Successfully removed VFs from PF.
2022-03-11 03:22:38 - sshInstance.py - [line:46] - root - INFO - virsh destroy VMtest1
2022-03-11 03:22:39 - sshInstance.py - [line:50] - root - INFO - Domain VMtest1 destroyed


2022-03-11 03:22:45 - test_network_sriov_stress.py - [line:276] - root - INFO - Successfully shut down the VM: VMtest1
2022-03-11 03:22:48 - sshInstance.py - [line:46] - root - INFO - virsh undefine VMtest1
2022-03-11 03:22:48 - sshInstance.py - [line:50] - root - INFO - Domain VMtest1 has been undefined


2022-03-11 03:22:53 - test_network_sriov_stress.py - [line:286] - root - INFO - Successfully removed the VM: VMtest1
2022-03-11 03:22:56 - sshInstance.py - [line:46] - root - INFO - virsh destroy VMtest2
2022-03-11 03:22:57 - sshInstance.py - [line:50] - root - INFO - Domain VMtest2 destroyed


2022-03-11 03:23:02 - test_network_sriov_stress.py - [line:276] - root - INFO - Successfully shut down the VM: VMtest2
2022-03-11 03:23:05 - sshInstance.py - [line:46] - root - INFO - virsh undefine VMtest2
2022-03-11 03:23:05 - sshInstance.py - [line:50] - root - INFO - Domain VMtest2 has been undefined


2022-03-11 03:23:10 - test_network_sriov_stress.py - [line:286] - root - INFO - Successfully removed the VM: VMtest2
2022-03-11 03:23:10 - test_network_sriov_stress.py - [line:389] - root - INFO - 

********************************************************Stress Disable and Enable for 4 time when VF is Assigned VM guest.*************************************************************


2022-03-11 03:23:13 - sshInstance.py - [line:46] - root - INFO - /opt/APP/utility/DPDK_install/dpdk-21.02/usertools/dpdk-devbind.py -s | grep ens785f2 | awk '{print $1;}'
2022-03-11 03:23:13 - sshInstance.py - [line:50] - root - INFO - 0000:4b:00.2

2022-03-11 03:23:18 - test_network_sriov_stress.py - [line:110] - root - INFO - pf_id was found: 0000:4b:00.2

2022-03-11 03:23:18 - test_network_sriov_stress.py - [line:397] - root - INFO - format_pf_id is 0000\:4b\:00.2

2022-03-11 03:23:18 - test_network_sriov_stress.py - [line:400] - root - INFO - pf_id_key: 4b
2022-03-11 03:23:24 - sshInstance.py - [line:94] - root - INFO - ifconfig ens785f2 192.168.10.10
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:23:27 - test_network_sriov_stress.py - [line:301] - root - INFO - Successfully assigned ip to ethernet: ens785f2
2022-03-11 03:23:30 - sshInstance.py - [line:46] - root - INFO - echo 4 > /sys/class/net/ens785f2/device/sriov_numvfs
2022-03-11 03:23:30 - sshInstance.py - [line:50] - root - INFO - 
2022-03-11 03:23:35 - test_network_sriov_stress.py - [line:124] - root - INFO - Successfully Virtualized PF to VFs.
2022-03-11 03:23:38 - sshInstance.py - [line:46] - root - INFO -  lspci | grep "Virtual" | awk '{print $1;}'list
2022-03-11 03:23:39 - sshInstance.py - [line:50] - root - INFO - 4b:11.0
4b:11.1
4b:11.2
4b:11.3

2022-03-11 03:23:44 - test_network_sriov_stress.py - [line:138] - root - INFO - Succssfully Gotten vf_id list:

2022-03-11 03:23:47 - sshInstance.py - [line:46] - root - INFO - modprobe vfio-pci

2022-03-11 03:23:47 - sshInstance.py - [line:50] - root - INFO - 
2022-03-11 03:23:55 - sshInstance.py - [line:46] - root - INFO - /opt/APP/utility/DPDK_install/dpdk-21.02/usertools/dpdk-devbind.py -b vfio-pci 4b:11.0
2022-03-11 03:23:56 - sshInstance.py - [line:50] - root - INFO - 
2022-03-11 03:24:01 - test_network_sriov_stress.py - [line:171] - root - INFO - Successfuly bound VF to vfio-pci
2022-03-11 03:24:04 - sshInstance.py - [line:46] - root - INFO - /opt/APP/utility/DPDK_install/dpdk-21.02/usertools/dpdk-devbind.py -b vfio-pci 4b:11.1
2022-03-11 03:24:05 - sshInstance.py - [line:50] - root - INFO - 
2022-03-11 03:24:10 - test_network_sriov_stress.py - [line:171] - root - INFO - Successfuly bound VF to vfio-pci
2022-03-11 03:24:10 - test_network_sriov_stress.py - [line:421] - root - INFO - lspci -Dnn | grep 4b:11.1
2022-03-11 03:24:13 - sshInstance.py - [line:46] - root - INFO - lspci -Dnn | grep 4b:11.1
2022-03-11 03:24:13 - sshInstance.py - [line:50] - root - INFO - 0000:4b:11.1 Ethernet controller [0200]: Intel Corporation Ethernet Adaptive Virtual Function [8086:1889] (rev 02)

2022-03-11 03:24:18 - test_network_sriov_stress.py - [line:432] - root - INFO - Successfully get vf device id: 8086:1889
2022-03-11 03:24:21 - sshInstance.py - [line:46] - root - INFO - virsh net-info default | grep "Active" | awk '{print $2;}'
2022-03-11 03:24:21 - sshInstance.py - [line:50] - root - INFO - yes

2022-03-11 03:24:26 - test_network_sriov_stress.py - [line:197] - root - INFO - Default network is active? yes

2022-03-11 03:24:30 - sshInstance.py - [line:71] - root - INFO - virt-install --name VMtest1 --ram 4096 --cpu host --vcpuss=5 --network network=default --hostdev pci_0000_4b_11_0 --boot hd --disk path=/hhome/image/uvm1.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5900  --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
ERROR    Requested operation is not valid: PCI device 0000:4b:11.0 is in use by driver QEMU, domain VMtest1
Domain installation does not appear to have been successful.
If it was, you can restart your domain by running:
  virsh --connect qemu:///system start VMtest1
otherwise, please restart your installation.
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:24:33 - test_network_sriov_stress.py - [line:229] - root - ERROR - Failed to create a VM: virt-install --name VMtest1 --ram 4096 --cpu host --vcpuss=5 --network network=default --hostdev pci_0000_4b_11_0 --boot hd --disk path=/hhome/image/uvm1.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5900  --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
ERROR    Requested operation is not valid: PCI device 0000:4b:11.0 is in use by driver QEMU, domain VMtest1
Domain installation does not appear to have been successful.
If it was, you can restart your domain by running:
  virsh --connect qemu:///system start VMtest1
otherwise, please restart your installation.
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:24:33 - test_network_sriov_stress.py - [line:230] - root - ERROR - Failed to create a VM: 
2022-03-11 03:24:37 - sshInstance.py - [line:71] - root - INFO - virt-install --name VMtest1 --ram 4096 --cpu host --vcpuss=5 --network network=default --hostdev pci_0000_4b_11_0 --boot hd --disk path=/hhome/image/uvm1.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5900  --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
2022-03-11 03:24:40 - sshInstance.py - [line:71] - root - INFO - Domain creation completed.
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:24:43 - test_network_sriov_stress.py - [line:232] - root - INFO - Successfully created a VM: VMtest1
2022-03-11 03:24:47 - sshInstance.py - [line:71] - root - INFO - virt-install --name VMtest2 --ram 4096 --cpu host --vcpuss=5 --network network=default --hostdev pci_0000_4b_11_1 --boot hd --disk path=/hhome/image/uvm2.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5901  --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
ERROR    Requested operation is not valid: PCI device 0000:4b:11.1 is in use by driver QEMU, domain VMtest2
Domain installation does not appear to have been successful.
If it was, you can restart your domain by running:
  virsh --connect qemu:///system start VMtest2
otherwise, please restart your installation.
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:24:50 - test_network_sriov_stress.py - [line:229] - root - ERROR - Failed to create a VM: virt-install --name VMtest2 --ram 4096 --cpu host --vcpuss=5 --network network=default --hostdev pci_0000_4b_11_1 --boot hd --disk path=/hhome/image/uvm2.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5901  --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
ERROR    Requested operation is not valid: PCI device 0000:4b:11.1 is in use by driver QEMU, domain VMtest2
Domain installation does not appear to have been successful.
If it was, you can restart your domain by running:
  virsh --connect qemu:///system start VMtest2
otherwise, please restart your installation.
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:24:50 - test_network_sriov_stress.py - [line:230] - root - ERROR - Failed to create a VM: 
2022-03-11 03:24:54 - sshInstance.py - [line:71] - root - INFO - virt-install --name VMtest2 --ram 4096 --cpu host --vcpuss=5 --network network=default --hostdev pci_0000_4b_11_1 --boot hd --disk path=/hhome/image/uvm2.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5901  --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
2022-03-11 03:24:57 - sshInstance.py - [line:71] - root - INFO - Domain creation completed.
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:25:00 - test_network_sriov_stress.py - [line:232] - root - INFO - Successfully created a VM: VMtest2
2022-03-11 03:25:03 - sshInstance.py - [line:46] - root - INFO - virsh domifaddr VMtest1 | grep '192.168' | awk '{print $4;}'
2022-03-11 03:25:03 - sshInstance.py - [line:50] - root - INFO - 
2022-03-11 03:25:08 - test_network_sriov_stress.py - [line:249] - root - ERROR - not found IP of VMtest1
2022-03-11 03:26:11 - sshInstance.py - [line:46] - root - INFO - virsh domifaddr VMtest1 | grep '192.168' | awk '{print $4;}'
2022-03-11 03:26:11 - sshInstance.py - [line:50] - root - INFO - 192.168.122.73/24

2022-03-11 03:26:16 - test_network_sriov_stress.py - [line:244] - root - INFO - Successfully got VMtest1 IP: 192.168.122.73/24

2022-03-11 03:26:19 - sshInstance.py - [line:46] - root - INFO - virsh domifaddr VMtest2 | grep '192.168' | awk '{print $4;}'
2022-03-11 03:26:19 - sshInstance.py - [line:50] - root - INFO - 192.168.122.250/24

2022-03-11 03:26:24 - test_network_sriov_stress.py - [line:244] - root - INFO - Successfully got VMtest2 IP: 192.168.122.250/24

2022-03-11 03:26:24 - test_network_sriov_stress.py - [line:449] - root - INFO - ip of VM1: 192.168.122.73
2022-03-11 03:26:24 - test_network_sriov_stress.py - [line:450] - root - INFO - ip of VM1: 192.168.122.250
2022-03-11 03:26:28 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.122.73
PING 192.168.122.73 (192.168.122.73) 56(84) bytes of data.
64 bytes from 192.168.122.73: icmp_seq=1 ttl=64 time=0.188 ms
2022-03-11 03:26:29 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.73: icmp_seq=2 ttl=64 time=0.138 ms
2022-03-11 03:26:30 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.73: icmp_seq=3 ttl=64 time=0.083 ms
2022-03-11 03:26:31 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.73: icmp_seq=4 ttl=64 time=0.089 ms
2022-03-11 03:26:32 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.73: icmp_seq=5 ttl=64 time=0.124 ms

--- 192.168.122.73 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 87ms
rtt min/avg/max/mdev = 0.083/0.124/0.188/0.039 ms
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:26:41 - sshInstance.py - [line:94] - root - INFO - ssh root@192.168.122.73
root@192.168.122.73's password: 
2022-03-11 03:26:50 - sshInstance.py - [line:94] - root - INFO - 
Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
  System information as of Thu 10 Mar 2022 02:28:21 PM EST
  System load:  0.06               Processes:             138
  Usage of /:   37.9% of 19.07GB   Users logged in:       0
  Memory usage: 4%                 IPv4 address for ens3: 192.168.122.73
  Swap usage:   0%

372 updates can be installed immediately.
227 of these updates are security updates.
To see these additional updates run: apt list --upgradable

The list of available updates is more than a week old.
To check for new updates run: sudo apt update
Last login: Thu Mar 10 14:22:51 2022 from 192.168.122.1
root@localhost:~# 
2022-03-11 03:26:59 - sshInstance.py - [line:94] - root - INFO - lspci -Dnn | grep "8086:1889" --color=never| awk '{print $1;}' [A        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [K            '
0000:00:05.0
root@localhost:~# 
2022-03-11 03:27:05 - test_network_sriov_stress.py - [line:340] - root - INFO - successfully got vf pci id of vm1: 0000:00:05.0
2022-03-11 03:27:11 - sshInstance.py - [line:94] - root - INFO - ls /sys/bus/pci/devices/0000\:00\:05.0/net/ --color=never
ens5
root@localhost:~# 
2022-03-11 03:27:16 - test_network_sriov_stress.py - [line:354] - root - INFO - Successfully got vf device name of vm1: ens5
2022-03-11 03:27:22 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 192.168.10.12
root@localhost:~# 
2022-03-11 03:27:25 - test_network_sriov_stress.py - [line:301] - root - INFO - Successfully assigned ip to ethernet: ens5
2022-03-11 03:27:31 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 down
root@localhost:~# 
2022-03-11 03:27:40 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 hw ether 1234567890ac
root@localhost:~# 
2022-03-11 03:27:49 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 up
root@localhost:~# 
2022-03-11 03:27:58 - sshInstance.py - [line:94] - root - INFO - nmcli device connect ens5
Command 'nmcli' not found, but can be installed with:
apt install network-manager
root@localhost:~# 
2022-03-11 03:28:07 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5
ens5: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.10.12  netmask 255.255.255.0  broadcast 192.168.10.255
        inet6 fe80::1034:56ff:fe78:90ac  prefixlen 64  scopeid 0x20<link>
        ether 12:34:56:78:90:ac  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 9  bytes 766 (766.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
root@localhost:~# 
2022-03-11 03:28:16 - sshInstance.py - [line:94] - root - INFO - exit
logout
Connection to 192.168.122.73 closed.
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:28:23 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.122.250
PING 192.168.122.250 (192.168.122.250) 56(84) bytes of data.
64 bytes from 192.168.122.250: icmp_seq=1 ttl=64 time=0.144 ms
2022-03-11 03:28:24 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.250: icmp_seq=2 ttl=64 time=0.092 ms
2022-03-11 03:28:25 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.250: icmp_seq=3 ttl=64 time=0.076 ms
2022-03-11 03:28:26 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.250: icmp_seq=4 ttl=64 time=0.085 ms
2022-03-11 03:28:27 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.250: icmp_seq=5 ttl=64 time=0.070 ms

--- 192.168.122.250 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 81ms
rtt min/avg/max/mdev = 0.070/0.093/0.144/0.027 ms
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:28:36 - sshInstance.py - [line:94] - root - INFO - ssh root@192.168.122.250
root@192.168.122.250's password: 
2022-03-11 03:28:45 - sshInstance.py - [line:94] - root - INFO - 
Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
  System information as of Thu 10 Mar 2022 02:30:16 PM EST
  System load:  0.01               Processes:             135
  Usage of /:   37.9% of 19.07GB   Users logged in:       0
  Memory usage: 4%                 IPv4 address for ens3: 192.168.122.250
  Swap usage:   0%

372 updates can be installed immediately.
227 of these updates are security updates.
To see these additional updates run: apt list --upgradable

The list of available updates is more than a week old.
To check for new updates run: sudo apt update
Last login: Thu Mar 10 14:20:34 2022 from 192.168.122.1
root@localhost:~# 
2022-03-11 03:28:54 - sshInstance.py - [line:94] - root - INFO - lspci -Dnn | grep "8086:1889" --color=never| awk '{print $1;}' [A        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [K            '
0000:00:05.0
root@localhost:~# 
2022-03-11 03:29:00 - test_network_sriov_stress.py - [line:340] - root - INFO - successfully got vf pci id of vm1: 0000:00:05.0
2022-03-11 03:29:06 - sshInstance.py - [line:94] - root - INFO - ls /sys/bus/pci/devices/0000\:00\:05.0/net/ --color=never
ens5
root@localhost:~# 
2022-03-11 03:29:11 - test_network_sriov_stress.py - [line:354] - root - INFO - Successfully got vf device name of vm1: ens5
2022-03-11 03:29:17 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 192.168.10.13
root@localhost:~# 
2022-03-11 03:29:20 - test_network_sriov_stress.py - [line:301] - root - INFO - Successfully assigned ip to ethernet: ens5
2022-03-11 03:29:26 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 down
root@localhost:~# 
2022-03-11 03:29:35 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 hw ether 1234567890ab
root@localhost:~# 
2022-03-11 03:29:44 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 up
root@localhost:~# 
2022-03-11 03:29:54 - sshInstance.py - [line:94] - root - INFO - nmcli device connect ens5
Command 'nmcli' not found, but can be installed with:
apt install network-manager
root@localhost:~# 
2022-03-11 03:30:03 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5
ens5: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.10.13  netmask 255.255.255.0  broadcast 192.168.10.255
        inet6 fe80::1034:56ff:fe78:90ab  prefixlen 64  scopeid 0x20<link>
        ether 12:34:56:78:90:ab  txqueuelen 1000  (Ethernet)
        RX packets 3  bytes 1038 (1.0 KB)
        RX errors 0  dropped 2  overruns 0  frame 0
        TX packets 9  bytes 766 (766.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
root@localhost:~# 
2022-03-11 03:30:12 - sshInstance.py - [line:94] - root - INFO - exit
logout
Connection to 192.168.122.250 closed.
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:30:15 - test_network_sriov_stress.py - [line:468] - root - INFO - Ping two vms IP from SUT:
2022-03-11 03:30:19 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.10.12
PING 192.168.10.12 (192.168.10.12) 56(84) bytes of data.
64 bytes from 192.168.10.12: icmp_seq=1 ttl=64 time=0.193 ms
2022-03-11 03:30:20 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=2 ttl=64 time=0.074 ms
2022-03-11 03:30:21 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=3 ttl=64 time=0.052 ms
2022-03-11 03:30:22 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=4 ttl=64 time=0.050 ms
2022-03-11 03:30:23 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=5 ttl=64 time=0.050 ms

--- 192.168.10.12 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 77ms
rtt min/avg/max/mdev = 0.050/0.083/0.193/0.056 ms
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:30:30 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.10.13
PING 192.168.10.13 (192.168.10.13) 56(84) bytes of data.
64 bytes from 192.168.10.13: icmp_seq=1 ttl=64 time=4.57 ms
2022-03-11 03:30:31 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=2 ttl=64 time=0.064 ms
2022-03-11 03:30:32 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=3 ttl=64 time=0.049 ms
2022-03-11 03:30:33 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=4 ttl=64 time=0.037 ms
2022-03-11 03:30:34 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=5 ttl=64 time=0.036 ms

--- 192.168.10.13 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 70ms
rtt min/avg/max/mdev = 0.036/0.951/4.572/1.810 ms
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:30:37 - test_network_sriov_stress.py - [line:476] - root - INFO - ping VMtest2 IP from VMtest1
2022-03-11 03:30:41 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.122.73
PING 192.168.122.73 (192.168.122.73) 56(84) bytes of data.
64 bytes from 192.168.122.73: icmp_seq=1 ttl=64 time=0.212 ms
2022-03-11 03:30:42 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.73: icmp_seq=2 ttl=64 time=0.151 ms
2022-03-11 03:30:43 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.73: icmp_seq=3 ttl=64 time=0.085 ms
2022-03-11 03:30:44 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.73: icmp_seq=4 ttl=64 time=0.084 ms
2022-03-11 03:30:45 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.73: icmp_seq=5 ttl=64 time=0.074 ms

--- 192.168.122.73 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 126ms
rtt min/avg/max/mdev = 0.074/0.121/0.212/0.053 ms
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:30:54 - sshInstance.py - [line:94] - root - INFO - ssh root@192.168.122.73
root@192.168.122.73's password: 
2022-03-11 03:31:03 - sshInstance.py - [line:94] - root - INFO - 
Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
  System information as of Thu 10 Mar 2022 02:32:33 PM EST
  System load:  0.0                Processes:             131
  Usage of /:   37.9% of 19.07GB   Users logged in:       0
  Memory usage: 4%                 IPv4 address for ens3: 192.168.122.73
  Swap usage:   0%                 IPv4 address for ens5: 192.168.10.12

372 updates can be installed immediately.
227 of these updates are security updates.
To see these additional updates run: apt list --upgradable

The list of available updates is more than a week old.
To check for new updates run: sudo apt update
Last login: Thu Mar 10 14:28:22 2022 from 192.168.122.1
root@localhost:~# 
2022-03-11 03:31:10 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.10.13
PING 192.168.10.13 (192.168.10.13) 56(84) bytes of data.
64 bytes from 192.168.10.13: icmp_seq=1 ttl=64 time=3.89 ms
2022-03-11 03:31:11 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=2 ttl=64 time=0.088 ms
2022-03-11 03:31:12 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=3 ttl=64 time=0.082 ms
2022-03-11 03:31:13 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=4 ttl=64 time=0.058 ms
2022-03-11 03:31:14 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=5 ttl=64 time=0.067 ms

--- 192.168.10.13 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4074ms
rtt min/avg/max/mdev = 0.058/0.836/3.887/1.525 ms
root@localhost:~# 
2022-03-11 03:31:54 - sshInstance.py - [line:94] - root - INFO - exit
logout
Connection to 192.168.122.73 closed.
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:31:57 - test_network_sriov_stress.py - [line:484] - root - INFO - ping VMtest2 IP from VMtest1
2022-03-11 03:32:01 - sshInstance.py - [line:71] - root - INFO - ping -c 5 -I 192.168.10.10 192.168.10.11
PING 192.168.10.11 (192.168.10.11) from 192.168.10.10 : 56(84) bytes of data.
64 bytes from 192.168.10.11: icmp_seq=1 ttl=64 time=0.279 ms
2022-03-11 03:32:02 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=2 ttl=64 time=0.202 ms
2022-03-11 03:32:03 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=3 ttl=64 time=0.178 ms
2022-03-11 03:32:04 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=4 ttl=64 time=0.185 ms
2022-03-11 03:32:05 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=5 ttl=64 time=0.183 ms

--- 192.168.10.11 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 91ms
rtt min/avg/max/mdev = 0.178/0.205/0.279/0.039 ms
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:32:11 - sshInstance.py - [line:46] - root - INFO - echo 0 > /sys/class/net/ens785f2/device/sriov_numvfs
2022-03-11 03:32:14 - sshInstance.py - [line:50] - root - INFO - 
2022-03-11 03:32:19 - test_network_sriov_stress.py - [line:263] - root - INFO - Successfully removed VFs from PF.
2022-03-11 03:32:22 - sshInstance.py - [line:46] - root - INFO - virsh destroy VMtest1
2022-03-11 03:32:23 - sshInstance.py - [line:50] - root - INFO - Domain VMtest1 destroyed


2022-03-11 03:32:28 - test_network_sriov_stress.py - [line:276] - root - INFO - Successfully shut down the VM: VMtest1
2022-03-11 03:32:31 - sshInstance.py - [line:46] - root - INFO - virsh undefine VMtest1
2022-03-11 03:32:31 - sshInstance.py - [line:50] - root - INFO - Domain VMtest1 has been undefined


2022-03-11 03:32:36 - test_network_sriov_stress.py - [line:286] - root - INFO - Successfully removed the VM: VMtest1
2022-03-11 03:32:39 - sshInstance.py - [line:46] - root - INFO - virsh destroy VMtest2
2022-03-11 03:32:40 - sshInstance.py - [line:50] - root - INFO - Domain VMtest2 destroyed


2022-03-11 03:32:45 - test_network_sriov_stress.py - [line:276] - root - INFO - Successfully shut down the VM: VMtest2
2022-03-11 03:32:48 - sshInstance.py - [line:46] - root - INFO - virsh undefine VMtest2
2022-03-11 03:32:48 - sshInstance.py - [line:50] - root - INFO - Domain VMtest2 has been undefined


2022-03-11 03:32:53 - test_network_sriov_stress.py - [line:286] - root - INFO - Successfully removed the VM: VMtest2
2022-03-11 03:32:53 - test_network_sriov_stress.py - [line:389] - root - INFO - 

********************************************************Stress Disable and Enable for 5 time when VF is Assigned VM guest.*************************************************************


2022-03-11 03:32:56 - sshInstance.py - [line:46] - root - INFO - /opt/APP/utility/DPDK_install/dpdk-21.02/usertools/dpdk-devbind.py -s | grep ens785f2 | awk '{print $1;}'
2022-03-11 03:32:57 - sshInstance.py - [line:50] - root - INFO - 0000:4b:00.2

2022-03-11 03:33:02 - test_network_sriov_stress.py - [line:110] - root - INFO - pf_id was found: 0000:4b:00.2

2022-03-11 03:33:02 - test_network_sriov_stress.py - [line:397] - root - INFO - format_pf_id is 0000\:4b\:00.2

2022-03-11 03:33:02 - test_network_sriov_stress.py - [line:400] - root - INFO - pf_id_key: 4b
2022-03-11 03:33:08 - sshInstance.py - [line:94] - root - INFO - ifconfig ens785f2 192.168.10.10
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:33:11 - test_network_sriov_stress.py - [line:301] - root - INFO - Successfully assigned ip to ethernet: ens785f2
2022-03-11 03:33:14 - sshInstance.py - [line:46] - root - INFO - echo 4 > /sys/class/net/ens785f2/device/sriov_numvfs
2022-03-11 03:33:14 - sshInstance.py - [line:50] - root - INFO - 
2022-03-11 03:33:19 - test_network_sriov_stress.py - [line:124] - root - INFO - Successfully Virtualized PF to VFs.
2022-03-11 03:33:22 - sshInstance.py - [line:46] - root - INFO -  lspci | grep "Virtual" | awk '{print $1;}'list
2022-03-11 03:33:22 - sshInstance.py - [line:50] - root - INFO - 4b:11.0
4b:11.1
4b:11.2
4b:11.3

2022-03-11 03:33:27 - test_network_sriov_stress.py - [line:138] - root - INFO - Succssfully Gotten vf_id list:

2022-03-11 03:33:30 - sshInstance.py - [line:46] - root - INFO - modprobe vfio-pci

2022-03-11 03:33:30 - sshInstance.py - [line:50] - root - INFO - 
2022-03-11 03:33:38 - sshInstance.py - [line:46] - root - INFO - /opt/APP/utility/DPDK_install/dpdk-21.02/usertools/dpdk-devbind.py -b vfio-pci 4b:11.0
2022-03-11 03:33:39 - sshInstance.py - [line:50] - root - INFO - 
2022-03-11 03:33:44 - test_network_sriov_stress.py - [line:171] - root - INFO - Successfuly bound VF to vfio-pci
2022-03-11 03:33:47 - sshInstance.py - [line:46] - root - INFO - /opt/APP/utility/DPDK_install/dpdk-21.02/usertools/dpdk-devbind.py -b vfio-pci 4b:11.1
2022-03-11 03:33:48 - sshInstance.py - [line:50] - root - INFO - 
2022-03-11 03:33:53 - test_network_sriov_stress.py - [line:171] - root - INFO - Successfuly bound VF to vfio-pci
2022-03-11 03:33:53 - test_network_sriov_stress.py - [line:421] - root - INFO - lspci -Dnn | grep 4b:11.1
2022-03-11 03:33:56 - sshInstance.py - [line:46] - root - INFO - lspci -Dnn | grep 4b:11.1
2022-03-11 03:33:56 - sshInstance.py - [line:50] - root - INFO - 0000:4b:11.1 Ethernet controller [0200]: Intel Corporation Ethernet Adaptive Virtual Function [8086:1889] (rev 02)

2022-03-11 03:34:01 - test_network_sriov_stress.py - [line:432] - root - INFO - Successfully get vf device id: 8086:1889
2022-03-11 03:34:04 - sshInstance.py - [line:46] - root - INFO - virsh net-info default | grep "Active" | awk '{print $2;}'
2022-03-11 03:34:04 - sshInstance.py - [line:50] - root - INFO - yes

2022-03-11 03:34:09 - test_network_sriov_stress.py - [line:197] - root - INFO - Default network is active? yes

2022-03-11 03:34:13 - sshInstance.py - [line:71] - root - INFO - virt-install --name VMtest1 --ram 4096 --cpu host --vcpuss=5 --network network=default --hostdev pci_0000_4b_11_0 --boot hd --disk path=/hhome/image/uvm1.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5900  --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
ERROR    Requested operation is not valid: PCI device 0000:4b:11.0 is in use by driver QEMU, domain VMtest1
Domain installation does not appear to have been successful.
If it was, you can restart your domain by running:
  virsh --connect qemu:///system start VMtest1
otherwise, please restart your installation.
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:34:16 - test_network_sriov_stress.py - [line:229] - root - ERROR - Failed to create a VM: virt-install --name VMtest1 --ram 4096 --cpu host --vcpuss=5 --network network=default --hostdev pci_0000_4b_11_0 --boot hd --disk path=/hhome/image/uvm1.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5900  --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
ERROR    Requested operation is not valid: PCI device 0000:4b:11.0 is in use by driver QEMU, domain VMtest1
Domain installation does not appear to have been successful.
If it was, you can restart your domain by running:
  virsh --connect qemu:///system start VMtest1
otherwise, please restart your installation.
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:34:16 - test_network_sriov_stress.py - [line:230] - root - ERROR - Failed to create a VM: 
2022-03-11 03:34:20 - sshInstance.py - [line:71] - root - INFO - virt-install --name VMtest1 --ram 4096 --cpu host --vcpuss=5 --network network=default --hostdev pci_0000_4b_11_0 --boot hd --disk path=/hhome/image/uvm1.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5900  --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
2022-03-11 03:34:23 - sshInstance.py - [line:71] - root - INFO - Domain creation completed.
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:34:26 - test_network_sriov_stress.py - [line:232] - root - INFO - Successfully created a VM: VMtest1
2022-03-11 03:34:30 - sshInstance.py - [line:71] - root - INFO - virt-install --name VMtest2 --ram 4096 --cpu host --vcpuss=5 --network network=default --hostdev pci_0000_4b_11_1 --boot hd --disk path=/hhome/image/uvm2.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5901  --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
ERROR    Requested operation is not valid: PCI device 0000:4b:11.1 is in use by driver QEMU, domain VMtest2
Domain installation does not appear to have been successful.
If it was, you can restart your domain by running:
  virsh --connect qemu:///system start VMtest2
otherwise, please restart your installation.
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:34:33 - test_network_sriov_stress.py - [line:229] - root - ERROR - Failed to create a VM: virt-install --name VMtest2 --ram 4096 --cpu host --vcpuss=5 --network network=default --hostdev pci_0000_4b_11_1 --boot hd --disk path=/hhome/image/uvm2.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5901  --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
ERROR    Requested operation is not valid: PCI device 0000:4b:11.1 is in use by driver QEMU, domain VMtest2
Domain installation does not appear to have been successful.
If it was, you can restart your domain by running:
  virsh --connect qemu:///system start VMtest2
otherwise, please restart your installation.
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:34:33 - test_network_sriov_stress.py - [line:230] - root - ERROR - Failed to create a VM: 
2022-03-11 03:34:37 - sshInstance.py - [line:71] - root - INFO - virt-install --name VMtest2 --ram 4096 --cpu host --vcpuss=5 --network network=default --hostdev pci_0000_4b_11_1 --boot hd --disk path=/hhome/image/uvm2.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5901  --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
2022-03-11 03:34:40 - sshInstance.py - [line:71] - root - INFO - Domain creation completed.
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:34:43 - test_network_sriov_stress.py - [line:232] - root - INFO - Successfully created a VM: VMtest2
2022-03-11 03:34:46 - sshInstance.py - [line:46] - root - INFO - virsh domifaddr VMtest1 | grep '192.168' | awk '{print $4;}'
2022-03-11 03:34:46 - sshInstance.py - [line:50] - root - INFO - 
2022-03-11 03:34:51 - test_network_sriov_stress.py - [line:249] - root - ERROR - not found IP of VMtest1
2022-03-11 03:35:54 - sshInstance.py - [line:46] - root - INFO - virsh domifaddr VMtest1 | grep '192.168' | awk '{print $4;}'
2022-03-11 03:35:55 - sshInstance.py - [line:50] - root - INFO - 192.168.122.73/24

2022-03-11 03:36:00 - test_network_sriov_stress.py - [line:244] - root - INFO - Successfully got VMtest1 IP: 192.168.122.73/24

2022-03-11 03:36:03 - sshInstance.py - [line:46] - root - INFO - virsh domifaddr VMtest2 | grep '192.168' | awk '{print $4;}'
2022-03-11 03:36:03 - sshInstance.py - [line:50] - root - INFO - 192.168.122.250/24

2022-03-11 03:36:08 - test_network_sriov_stress.py - [line:244] - root - INFO - Successfully got VMtest2 IP: 192.168.122.250/24

2022-03-11 03:36:08 - test_network_sriov_stress.py - [line:449] - root - INFO - ip of VM1: 192.168.122.73
2022-03-11 03:36:08 - test_network_sriov_stress.py - [line:450] - root - INFO - ip of VM1: 192.168.122.250
2022-03-11 03:36:12 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.122.73
PING 192.168.122.73 (192.168.122.73) 56(84) bytes of data.
64 bytes from 192.168.122.73: icmp_seq=1 ttl=64 time=0.149 ms
2022-03-11 03:36:13 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.73: icmp_seq=2 ttl=64 time=0.097 ms
2022-03-11 03:36:14 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.73: icmp_seq=3 ttl=64 time=0.097 ms
2022-03-11 03:36:15 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.73: icmp_seq=4 ttl=64 time=0.161 ms
2022-03-11 03:36:16 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.73: icmp_seq=5 ttl=64 time=0.087 ms

--- 192.168.122.73 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 108ms
rtt min/avg/max/mdev = 0.087/0.118/0.161/0.031 ms
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:36:25 - sshInstance.py - [line:94] - root - INFO - ssh root@192.168.122.73
root@192.168.122.73's password: 
2022-03-11 03:36:34 - sshInstance.py - [line:94] - root - INFO - 
Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
  System information as of Thu 10 Mar 2022 02:38:05 PM EST
  System load:  0.03               Processes:             136
  Usage of /:   38.0% of 19.07GB   Users logged in:       0
  Memory usage: 4%                 IPv4 address for ens3: 192.168.122.73
  Swap usage:   0%

372 updates can be installed immediately.
227 of these updates are security updates.
To see these additional updates run: apt list --upgradable

The list of available updates is more than a week old.
To check for new updates run: sudo apt update
Last login: Thu Mar 10 14:32:34 2022 from 192.168.122.1
root@localhost:~# 
2022-03-11 03:36:43 - sshInstance.py - [line:94] - root - INFO - lspci -Dnn | grep "8086:1889" --color=never| awk '{print $1;}' [A        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [K            '
0000:00:05.0
root@localhost:~# 
2022-03-11 03:36:49 - test_network_sriov_stress.py - [line:340] - root - INFO - successfully got vf pci id of vm1: 0000:00:05.0
2022-03-11 03:36:55 - sshInstance.py - [line:94] - root - INFO - ls /sys/bus/pci/devices/0000\:00\:05.0/net/ --color=never
ens5
root@localhost:~# 
2022-03-11 03:37:00 - test_network_sriov_stress.py - [line:354] - root - INFO - Successfully got vf device name of vm1: ens5
2022-03-11 03:37:06 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 192.168.10.12
root@localhost:~# 
2022-03-11 03:37:09 - test_network_sriov_stress.py - [line:301] - root - INFO - Successfully assigned ip to ethernet: ens5
2022-03-11 03:37:15 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 down
root@localhost:~# 
2022-03-11 03:37:24 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 hw ether 1234567890ac
root@localhost:~# 
2022-03-11 03:37:33 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 up
root@localhost:~# 
2022-03-11 03:37:42 - sshInstance.py - [line:94] - root - INFO - nmcli device connect ens5
Command 'nmcli' not found, but can be installed with:
apt install network-manager
root@localhost:~# 
2022-03-11 03:37:51 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5
ens5: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.10.12  netmask 255.255.255.0  broadcast 192.168.10.255
        inet6 fe80::1034:56ff:fe78:90ac  prefixlen 64  scopeid 0x20<link>
        ether 12:34:56:78:90:ac  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 9  bytes 766 (766.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
root@localhost:~# 
2022-03-11 03:38:00 - sshInstance.py - [line:94] - root - INFO - exit
logout
Connection to 192.168.122.73 closed.
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:38:07 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.122.250
PING 192.168.122.250 (192.168.122.250) 56(84) bytes of data.
64 bytes from 192.168.122.250: icmp_seq=1 ttl=64 time=0.189 ms
2022-03-11 03:38:08 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.250: icmp_seq=2 ttl=64 time=0.100 ms
2022-03-11 03:38:09 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.250: icmp_seq=3 ttl=64 time=0.129 ms
2022-03-11 03:38:10 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.250: icmp_seq=4 ttl=64 time=0.125 ms
2022-03-11 03:38:11 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.250: icmp_seq=5 ttl=64 time=0.074 ms

--- 192.168.122.250 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 102ms
rtt min/avg/max/mdev = 0.074/0.123/0.189/0.039 ms
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:38:20 - sshInstance.py - [line:94] - root - INFO - ssh root@192.168.122.250
root@192.168.122.250's password: 
2022-03-11 03:38:29 - sshInstance.py - [line:94] - root - INFO - 
Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
  System information as of Thu 10 Mar 2022 02:40:00 PM EST
  System load:  0.0                Processes:             137
  Usage of /:   37.9% of 19.07GB   Users logged in:       0
  Memory usage: 4%                 IPv4 address for ens3: 192.168.122.250
  Swap usage:   0%

372 updates can be installed immediately.
227 of these updates are security updates.
To see these additional updates run: apt list --upgradable

The list of available updates is more than a week old.
To check for new updates run: sudo apt update
Last login: Thu Mar 10 14:30:16 2022 from 192.168.122.1
root@localhost:~# 
2022-03-11 03:38:38 - sshInstance.py - [line:94] - root - INFO - lspci -Dnn | grep "8086:1889" --color=never| awk '{print $1;}' [A        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [K            '
0000:00:05.0
root@localhost:~# 
2022-03-11 03:38:44 - test_network_sriov_stress.py - [line:340] - root - INFO - successfully got vf pci id of vm1: 0000:00:05.0
2022-03-11 03:38:50 - sshInstance.py - [line:94] - root - INFO - ls /sys/bus/pci/devices/0000\:00\:05.0/net/ --color=never
ens5
root@localhost:~# 
2022-03-11 03:38:55 - test_network_sriov_stress.py - [line:354] - root - INFO - Successfully got vf device name of vm1: ens5
2022-03-11 03:39:01 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 192.168.10.13
root@localhost:~# 
2022-03-11 03:39:04 - test_network_sriov_stress.py - [line:301] - root - INFO - Successfully assigned ip to ethernet: ens5
2022-03-11 03:39:10 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 down
root@localhost:~# 
2022-03-11 03:39:19 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 hw ether 1234567890ab
root@localhost:~# 
2022-03-11 03:39:28 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5 up
root@localhost:~# 
2022-03-11 03:39:37 - sshInstance.py - [line:94] - root - INFO - nmcli device connect ens5
Command 'nmcli' not found, but can be installed with:
apt install network-manager
root@localhost:~# 
2022-03-11 03:39:46 - sshInstance.py - [line:94] - root - INFO - ifconfig ens5
ens5: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.10.13  netmask 255.255.255.0  broadcast 192.168.10.255
        inet6 fe80::1034:56ff:fe78:90ab  prefixlen 64  scopeid 0x20<link>
        ether 12:34:56:78:90:ab  txqueuelen 1000  (Ethernet)
        RX packets 2  bytes 692 (692.0 B)
        RX errors 0  dropped 1  overruns 0  frame 0
        TX packets 9  bytes 766 (766.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
root@localhost:~# 
2022-03-11 03:39:55 - sshInstance.py - [line:94] - root - INFO - exit
logout
Connection to 192.168.122.250 closed.
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:39:58 - test_network_sriov_stress.py - [line:468] - root - INFO - Ping two vms IP from SUT:
2022-03-11 03:40:02 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.10.12
PING 192.168.10.12 (192.168.10.12) 56(84) bytes of data.
64 bytes from 192.168.10.12: icmp_seq=1 ttl=64 time=0.187 ms
2022-03-11 03:40:03 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=2 ttl=64 time=0.043 ms
2022-03-11 03:40:04 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=3 ttl=64 time=0.048 ms
2022-03-11 03:40:05 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=4 ttl=64 time=0.038 ms
2022-03-11 03:40:06 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=5 ttl=64 time=0.041 ms

--- 192.168.10.12 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 92ms
rtt min/avg/max/mdev = 0.038/0.071/0.187/0.058 ms
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:40:13 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.10.13
PING 192.168.10.13 (192.168.10.13) 56(84) bytes of data.
64 bytes from 192.168.10.13: icmp_seq=1 ttl=64 time=4.43 ms
2022-03-11 03:40:14 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=2 ttl=64 time=0.073 ms
2022-03-11 03:40:15 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=3 ttl=64 time=0.036 ms
2022-03-11 03:40:16 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=4 ttl=64 time=0.069 ms
2022-03-11 03:40:17 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=5 ttl=64 time=0.041 ms

--- 192.168.10.13 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 85ms
rtt min/avg/max/mdev = 0.036/0.928/4.425/1.748 ms
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:40:20 - test_network_sriov_stress.py - [line:476] - root - INFO - ping VMtest2 IP from VMtest1
2022-03-11 03:40:24 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.122.73
PING 192.168.122.73 (192.168.122.73) 56(84) bytes of data.
64 bytes from 192.168.122.73: icmp_seq=1 ttl=64 time=0.138 ms
2022-03-11 03:40:25 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.73: icmp_seq=2 ttl=64 time=0.137 ms
2022-03-11 03:40:26 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.73: icmp_seq=3 ttl=64 time=0.090 ms
2022-03-11 03:40:27 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.73: icmp_seq=4 ttl=64 time=0.092 ms
2022-03-11 03:40:28 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.73: icmp_seq=5 ttl=64 time=0.162 ms

--- 192.168.122.73 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 78ms
rtt min/avg/max/mdev = 0.090/0.123/0.162/0.031 ms
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:40:37 - sshInstance.py - [line:94] - root - INFO - ssh root@192.168.122.73
root@192.168.122.73's password: 
2022-03-11 03:40:46 - sshInstance.py - [line:94] - root - INFO - 
Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
  System information as of Thu 10 Mar 2022 02:42:18 PM EST
  System load:  0.0                Processes:             129
  Usage of /:   38.0% of 19.07GB   Users logged in:       0
  Memory usage: 4%                 IPv4 address for ens3: 192.168.122.73
  Swap usage:   0%                 IPv4 address for ens5: 192.168.10.12

372 updates can be installed immediately.
227 of these updates are security updates.
To see these additional updates run: apt list --upgradable

The list of available updates is more than a week old.
To check for new updates run: sudo apt update
Last login: Thu Mar 10 14:38:06 2022 from 192.168.122.1
root@localhost:~# 
2022-03-11 03:40:53 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.10.13
PING 192.168.10.13 (192.168.10.13) 56(84) bytes of data.
64 bytes from 192.168.10.13: icmp_seq=1 ttl=64 time=4.72 ms
2022-03-11 03:40:54 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=2 ttl=64 time=0.076 ms
2022-03-11 03:40:55 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=3 ttl=64 time=0.047 ms
2022-03-11 03:40:56 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=4 ttl=64 time=0.044 ms
2022-03-11 03:40:57 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=5 ttl=64 time=0.054 ms

--- 192.168.10.13 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4070ms
rtt min/avg/max/mdev = 0.044/0.988/4.723/1.867 ms
root@localhost:~# 
2022-03-11 03:41:37 - sshInstance.py - [line:94] - root - INFO - exit
logout
Connection to 192.168.122.73 closed.
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:41:40 - test_network_sriov_stress.py - [line:484] - root - INFO - ping VMtest2 IP from VMtest1
2022-03-11 03:41:44 - sshInstance.py - [line:71] - root - INFO - ping -c 5 -I 192.168.10.10 192.168.10.11
PING 192.168.10.11 (192.168.10.11) from 192.168.10.10 : 56(84) bytes of data.
64 bytes from 192.168.10.11: icmp_seq=1 ttl=64 time=0.246 ms
2022-03-11 03:41:45 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=2 ttl=64 time=0.153 ms
2022-03-11 03:41:46 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=3 ttl=64 time=0.184 ms
2022-03-11 03:41:47 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=4 ttl=64 time=0.198 ms
2022-03-11 03:41:48 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=5 ttl=64 time=0.175 ms

--- 192.168.10.11 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 108ms
rtt min/avg/max/mdev = 0.153/0.191/0.246/0.032 ms
[root@npx-cyp7-rh82 ~]# 
2022-03-11 03:41:54 - sshInstance.py - [line:46] - root - INFO - echo 0 > /sys/class/net/ens785f2/device/sriov_numvfs
2022-03-11 03:41:57 - sshInstance.py - [line:50] - root - INFO - 
2022-03-11 03:42:02 - test_network_sriov_stress.py - [line:263] - root - INFO - Successfully removed VFs from PF.
2022-03-11 03:42:05 - sshInstance.py - [line:46] - root - INFO - virsh destroy VMtest1
2022-03-11 03:42:06 - sshInstance.py - [line:50] - root - INFO - Domain VMtest1 destroyed


2022-03-11 03:42:11 - test_network_sriov_stress.py - [line:276] - root - INFO - Successfully shut down the VM: VMtest1
2022-03-11 03:42:14 - sshInstance.py - [line:46] - root - INFO - virsh undefine VMtest1
2022-03-11 03:42:14 - sshInstance.py - [line:50] - root - INFO - Domain VMtest1 has been undefined


2022-03-11 03:42:19 - test_network_sriov_stress.py - [line:286] - root - INFO - Successfully removed the VM: VMtest1
2022-03-11 03:42:22 - sshInstance.py - [line:46] - root - INFO - virsh destroy VMtest2
2022-03-11 03:42:23 - sshInstance.py - [line:50] - root - INFO - Domain VMtest2 destroyed


2022-03-11 03:42:28 - test_network_sriov_stress.py - [line:276] - root - INFO - Successfully shut down the VM: VMtest2
2022-03-11 03:42:31 - sshInstance.py - [line:46] - root - INFO - virsh undefine VMtest2
2022-03-11 03:42:31 - sshInstance.py - [line:50] - root - INFO - Domain VMtest2 has been undefined


2022-03-11 03:42:36 - test_network_sriov_stress.py - [line:286] - root - INFO - Successfully removed the VM: VMtest2
.2022-03-11 03:42:39 - sshInstance.py - [line:46] - root - INFO - virsh destroy VMtest2
2022-03-11 03:42:39 - sshInstance.py - [line:50] - root - INFO - error: failed to get domain 'VMtest2'


2022-03-11 03:42:44 - test_network_sriov_stress.py - [line:276] - root - INFO - Successfully shut down the VM: VMtest2
2022-03-11 03:42:47 - sshInstance.py - [line:46] - root - INFO - virsh undefine VMtest2
2022-03-11 03:42:47 - sshInstance.py - [line:50] - root - INFO - error: failed to get domain 'VMtest2'


2022-03-11 03:42:52 - test_network_sriov_stress.py - [line:289] - root - ERROR - Failed to removed the VM VMtest2
2022-03-11 03:42:55 - sshInstance.py - [line:46] - root - INFO - virsh destroy VMtest1
2022-03-11 03:42:55 - sshInstance.py - [line:50] - root - INFO - error: failed to get domain 'VMtest1'


2022-03-11 03:43:00 - test_network_sriov_stress.py - [line:276] - root - INFO - Successfully shut down the VM: VMtest1
2022-03-11 03:43:03 - sshInstance.py - [line:46] - root - INFO - virsh undefine VMtest1
2022-03-11 03:43:04 - sshInstance.py - [line:50] - root - INFO - error: failed to get domain 'VMtest1'


2022-03-11 03:43:09 - test_network_sriov_stress.py - [line:289] - root - ERROR - Failed to removed the VM VMtest1
2022-03-11 03:43:12 - sshInstance.py - [line:46] - root - INFO - echo 0 > /sys/class/net/ens785f2/device/sriov_numvfs
2022-03-11 03:43:12 - sshInstance.py - [line:50] - root - INFO - 
2022-03-11 03:43:20 - sshInstance.py - [line:46] - root - INFO - ifconfig ens785f2 0
2022-03-11 03:43:20 - sshInstance.py - [line:50] - root - INFO - 
2022-03-11 03:43:28 - sshInstance.py - [line:46] - root - INFO - dmesg > /tmp/dmesg.log
2022-03-11 03:43:28 - sshInstance.py - [line:50] - root - INFO - 
2022-03-11 03:43:33 - test_network_sriov_stress.py - [line:81] - root - INFO - generate dmesg.log file on sut /tmp/dmesg.log
2022-03-11 03:43:33 - test_network_sriov_stress.py - [line:83] - root - INFO - copy dmesg.log from sut1 /tmp dir to controller dir /root/testconfig/testlogs/2022-03-11/test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest[uvm1.img-uvm2.img]_02_52_42
2022-03-11 03:43:36 - sshInstance.py - [line:46] - root - INFO - rm -f /tmp/dmesg.log
2022-03-11 03:43:36 - sshInstance.py - [line:50] - root - INFO - 
2022-03-11 03:43:41 - test_network_sriov_stress.py - [line:85] - root - INFO - rm dmesg.log on sut1 /tmp dir
2022-03-11 03:43:41 - sshInstance.py - [line:39] - root - INFO - closing ssh client on SUT
2022-03-11 03:43:41 - NicTestFixture.py - [line:32] - root - INFO - ============================ end of run test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest[uvm1.img-uvm2.img] case ============================
                  [100%]

=============================== warnings summary ===============================
npxtester/Tests/testNics/test_iperf_client_4hour.py:69
  /root/.jenkins/workspace/cyp-test-nic-sriov-5times/npxtester/Tests/testNics/test_iperf_client_4hour.py:69: PytestUnknownMarkWarning: Unknown pytest.mark.iperf_tcp - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.iperf_tcp

npxtester/Tests/testNics/test_iperf_client_4hour.py:107
  /root/.jenkins/workspace/cyp-test-nic-sriov-5times/npxtester/Tests/testNics/test_iperf_client_4hour.py:107: PytestUnknownMarkWarning: Unknown pytest.mark.iperf_udp - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.iperf_udp

npxtester/Tests/testNics/test_mtu.py:69
  /root/.jenkins/workspace/cyp-test-nic-sriov-5times/npxtester/Tests/testNics/test_mtu.py:69: PytestUnknownMarkWarning: Unknown pytest.mark.two_sut_CVL - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.two_sut_CVL

npxtester/Tests/testNics/test_mtu.py:70
  /root/.jenkins/workspace/cyp-test-nic-sriov-5times/npxtester/Tests/testNics/test_mtu.py:70: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_mtu.py:109
  /root/.jenkins/workspace/cyp-test-nic-sriov-5times/npxtester/Tests/testNics/test_mtu.py:109: PytestUnknownMarkWarning: Unknown pytest.mark.two_sut_CVL - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.two_sut_CVL

npxtester/Tests/testNics/test_mtu.py:110
  /root/.jenkins/workspace/cyp-test-nic-sriov-5times/npxtester/Tests/testNics/test_mtu.py:110: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_network_sriov_stress.py:382
  /root/.jenkins/workspace/cyp-test-nic-sriov-5times/npxtester/Tests/testNics/test_network_sriov_stress.py:382: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_network_sriov_stress.py:383
  /root/.jenkins/workspace/cyp-test-nic-sriov-5times/npxtester/Tests/testNics/test_network_sriov_stress.py:383: PytestUnknownMarkWarning: Unknown pytest.mark.nicSriov5times - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSriov5times

npxtester/Tests/testNics/test_network_sriov_stress.py:500
  /root/.jenkins/workspace/cyp-test-nic-sriov-5times/npxtester/Tests/testNics/test_network_sriov_stress.py:500: PytestUnknownMarkWarning: Unknown pytest.mark.nicSriov2hours - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSriov2hours

npxtester/Tests/testNics/test_network_sriov_stress.py:799
  /root/.jenkins/workspace/cyp-test-nic-sriov-5times/npxtester/Tests/testNics/test_network_sriov_stress.py:799: PytestUnknownMarkWarning: Unknown pytest.mark.SriovVfIperf - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.SriovVfIperf

npxtester/Tests/testNics/test_nics.py:85
  /root/.jenkins/workspace/cyp-test-nic-sriov-5times/npxtester/Tests/testNics/test_nics.py:85: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:110
  /root/.jenkins/workspace/cyp-test-nic-sriov-5times/npxtester/Tests/testNics/test_nics.py:110: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:134
  /root/.jenkins/workspace/cyp-test-nic-sriov-5times/npxtester/Tests/testNics/test_nics.py:134: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:166
  /root/.jenkins/workspace/cyp-test-nic-sriov-5times/npxtester/Tests/testNics/test_nics.py:166: PytestUnknownMarkWarning: Unknown pytest.mark.driveInstall - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.driveInstall

npxtester/Tests/testNics/test_nics.py:206
  /root/.jenkins/workspace/cyp-test-nic-sriov-5times/npxtester/Tests/testNics/test_nics.py:206: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:232
  /root/.jenkins/workspace/cyp-test-nic-sriov-5times/npxtester/Tests/testNics/test_nics.py:232: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

-- Docs: https://docs.pytest.org/en/stable/warnings.html
- generated html file: file:///root/testconfig/testlogs/HtmlReport/nic_sriov5times_case_report.html -
========== 1 passed, 16 deselected, 16 warnings in 3059.05s (0:50:59) ==========
[8mha:////4LeiMYyvaTAszhIQ10pdIeDepJ20hGjpoNTSDQ54vTlOAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIi0SCEqNJabniBiY1xYt0Z2yGpeBFf4w8EIlGxxWpnm3m+YJkiHDlabA11jlLjMPjeTgsHjt3F84Atn7FhSuwNSjNI1qaeUXI2MKcoYSFgZajxnBzZDGvRqruqvCJbnXKcvoOA0ukbPKAQkzmrmL8wRthce52Yfp6/AhhDyFBu95/avQGMKkS1vgAAAA==[0m[Pipeline] }
[8mha:////4I/tRoD/KV6lJ76TTEJ8vusLD5mbZiDJ1z7tZBIwhHThAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOKCQKJBqWgtN7zAJMY4se6C7eBUvIiv8QcsIlGxxWpnm3m9YRk81OwNdpp6S6GxOLjR5IWJfX91nLDjCzZMgZ1GqZPkVp9mlBw1zClKWAhYaWocB0smwlp06qEqp8hU5+jzdxRQ2vYOTyhENkfl4xcmD5vb2Aamn+evAKZhiFDut7l2hw9mQDtDvgAAAA==[0m[Pipeline] // stage
[8mha:////4CAP7WQdC2FMpNczqgZSKdPjM+Zu6Lz59I8VEE3/AlUOAAAAph+LCAAAAAAAAP9tjTEOwjAQBM9BKWgpeYQjBB2iorXc8AITG+PEugv2haTiRXyNPxCIRMVWOyut5vmCMic4UPKycdgGzHWQXez91ORAqb1EGmRDZ1kTZopOajdosu44oyZ2MEcUsFCwdFhHygE9w0o15m6qaNBXJ07TtldQBHuDBwg1mdkk/sKYYH3tbSb8ef4KYOwYxI6h2G4+x/INOibpDcEAAAA=[0m[Pipeline] stage
[8mha:////4Db8sOlSRxZnVxb7xJBDSAteoF0PgC9lkQldzKIJFwYiAAAAph+LCAAAAAAAAP9tjTEOwjAUQ3+LOrAycohUwIiYukZZOEFoQkgb/V9+UtqJE3E17kBLJSY8WLYl673eUESGE7ETjcXWY6y96ELvpiQG4vYaaBANXURNGClYoeygyNhqqYqShUVZDisJa4t1oOjRJdjIRj90GTS68px42o4Scm/u8IRMTuSkOX3LyLC99SYS/jh/ATB2CfLDbrb9/Cw+2YMO4MIAAAA=[0m[Pipeline] { (collect logs)
[8mha:////4LN/0LMxu1hyybTBeE6CHwDx82bv/lkvxFk3cdgHTSuWAAAAox+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIikRJR0VppeIFJjLFj3Tm2g1PxIr7GHwhEomKL1c4283zBOgY4ctBoFfWGYmvQu1HPCzOH/uo4o+ULtkyRncJG5YY7dVqw4aRgSVHCSsBGUes4GtIJtsLKu6ycJF2dU5i/g4DSdAM8oBCzOcmQvjAF2N3GLjL9PH8FMPkEZb3/VO3favo5XL4AAAA=[0m[Pipeline] sh
+ sed -i $'s/\r//g' /root/.jenkins/workspace/cyp-test-nic-sriov-5times/npxtester/log_analyzer/log_analyzer.sh
[8mha:////4FgvIflgycWoJq7MsZycbV9d7HENRG2WSX14oF+Me/GuAAAApB+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIiAR2iSmu54QUmNsaJdWdsh6TiRXyNPxCIRMUWq51t5vmCZYpw5GixNdQ5So3D4Hs7LRw4dhfPA7Z8xoYpsTcozSBZm3pGydnAnKKEhYCVocZzcmQzrEWr7qryimx1ynH6DgJKp2/wgEJM5qxi/sIYYXPtdWL6ef4KYAwZyt32U/vwBq1seBO+AAAA[0m[Pipeline] sh
+ chmod a+x /root/.jenkins/workspace/cyp-test-nic-sriov-5times/npxtester/log_analyzer/log_analyzer.sh
[8mha:////4P37N80+NRbzyFlga2j1WDon6H/4VmnQwrRysqQpQX/3AAAApB+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIigagQVVrLDS8wsTFOrDtjOyQVL+Jr/IFAJCq2WO1sM88XLFOEI0eLraHOUWocBt/baeHAsbt4HrDlMzZMib1BaQbJ2tQzSs4G5hQlLASsDDWekyObYS1adVeVV2SrU47TdxBQOn2DBxRiMmcV8xfGCJtrrxPTz/NXAGPIUO62n9qHN+xdYwq+AAAA[0m[Pipeline] sh
+ sh /root/.jenkins/workspace/cyp-test-nic-sriov-5times/npxtester/log_analyzer/log_analyzer.sh
/root/.jenkins/workspace/cyp-test-nic-sriov-5times/npxtester/log_analyzer/log_analyzer.sh: line 7: [: too many arguments
[8mha:////4P3ih/vRXCzaXCkS1CQDBQXrb3I4EZcCr0NGGH+tKlR4AAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIigUSDqNJabniBiY1xYt0Z2yGpeBFf4w8EIlGxxWpnm3m+YJkiHDlabA11jlLjMPjeTgsHjt3F84Atn7FhSuwNSjNI1qaeUXI2MKcoYSFgZajxnBzZDGvRqruqvCJbnXKcvoOA0ukbPKAQkzmrmL8wRthce52Yfp6/AhhDyFDu9p/avgEjVhSLvgAAAA==[0m[Pipeline] }
[8mha:////4ISV0Ptmba0km8a80Gq+UClogdZnPDxCJXroOoeI5sULAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIiREGBqNJabniBiY1xYt0Z2yGpeBFf4w8EIlGxxWpnm3m+YJkiHDlabA11jlLjMPjeTgsHjt3F84Atn7FhSuwNSjNI1qaeUXI2MKcoYSFgZajxnBzZDGvRqruqvCJbnXKcvoOA0ukbPKAQkzmrmL8wRthce52Yfp6/AhhDyFDu9p/avgE81EHZvgAAAA==[0m[Pipeline] // stage
[8mha:////4LmA5LOJ1yGOO5YQ6RYn+/szIsYEkfxAArsUKSFs0Xa0AAAApB+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMIpoEBCVLRWGl5gEmOcWHfBvpBUvIiv8QciIlGx1c4083pjmSKOHJ1qLLWeUuVVF3o3PTVwbK+BB9XwRVVMiYNVpR1Kru1pxpLFYl6WY6GxslQFTp6cYK0b8zBFMOSKs8TJHTRyX9/xRKanspgoXxgjNre+Tky/zt8Axq4T5Nu9INt9AE/B/a+9AAAA[0m[Pipeline] }
[8mha:////4KVKO0UZMawo8C9tjXN4L1+ryWbU4zmvD2sDlsKoyYtXAAAAox+LCAAAAAAAAP9tjTESgjAQRT84FraWHiIUVI5jZZuh8QQRYgxkdjFZhMoTeTXvICMzVv7qv9e81xvrFHHk6FRrqfOUaq/6MLj5qZFjdw08qpYvqmZKHKyq7FhxY08LViwWy7IcK42NpTpw8uQEW92ahymCIVecJc7uoJH75o4nMj2XxUT5whSxuw1NYvp1/gYw9b0gL/eCrPwAXH35DL0AAAA=[0m[Pipeline] // node
[8mha:////4A8vvhfR6oegt+2XAbvn3Z0Fx6j3lb1QUh2abilCs+OGAAAAox+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMJBiA5R0VppeIFJjHFi3QX7QlLxIr7GH4iIRMVWO9PM641lijhydKqx1HpKlVdd6N301MCxvQYeVMMXVTElDlaVdii5tqcZSxaLeVmOhcbKUhU4eXKCtW7MwxTBkCvOEid30Mh9fccTmZ7KYqJ8YYzY3Po6Mf06fwMYu06Q77eCbPcBAx/ra70AAAA=[0m[Pipeline] End of Pipeline
Finished: SUCCESS
